---
title: SARS-CoV-2 Variants of Concern (VOC)
toc: true
---

<div class="row mb-4"><div class="col col-sm-8"><ul class="nav nav-tabs nav-justified">
  <li class="nav-item">
    <a class="nav-link active" href="#"><b>Variants of Concern (VOC)</b></a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="omicron">Omicron<br></a>
  </li>
</ul></div></div>

## Background and nomenclature for SARS CoV-2 variants

SARS-CoV-2 is an *RNA virus*. Such viruses are known for their fast mutation rates and their resilience to antiviral drugs.

A variant of virus can be thought of as a 'version' of a given virus. Variants have at least one mutation that makes them distinct from other 'versions' of that same virus. Multiple variants of a virus can circulate in a population at any given time.

Since the start of the COVID-19 pandemic in December 2019, multiple SARS-CoV-2 variants have emerged in different parts of the world. The research community and governmental health agencies closely monitor all new variants. However, some variants exhibit characteristics that *could* indicate that they *may* pose a greater health risk. These variants are naturally prioritised, and are monitored especially closely. The research community initially used nomenclature from [Pango](https://cov-lineages.org), [Nextstrain](https://nextstrain.org), and [GISAID](https://www.gisaid.org) to refer to different variants. However, in May 2021, the World Health Organization (WHO) proposed that [variants were named after the letters of the Greek alphabet](https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/). The Greek letters were then used, often alongside previous scientific nomenclature, in communications about different variants. As of December 2021, the latest veriant was named 'Omicron'.

[*The European Centre for Disease Control*](https://www.ecdc.europa.eu/en) place SARS-CoV-2 variants into one of three categories, although their assignment may change over time:

* **Variant of Concern (VOC)**
* **Variant of Interest (VOI)**
* **Variant under monitoring (VUM)**

##### Variant of concern

Variants are categorised as a *Variant of concern (VOC)* if there is clear evidence that the variant will have/is having a marked impact on the epidemiological situation in a population. This is likely to occur because the variant is more transmissable, causes more severe disease, and/or has a more detrimental effect on host immunity. There may also be research indicating that the variant has the potential for *vaccine escape* (i.e. there is a significant reduction in the neutralising effect of antibodies resulting from vaccination), or that the vaccine is less protective against the development of severe disease. A SARS-CoV-2 *VOC* may also have proven to be relatively more difficult to detect using standard diagnostic assays, making the spread more difficult to track.

As of December 2021, there are four *VOCs* in Sweden: **Beta**, **Gamma**, **Delta**, and **Omicron**.

##### Variant of Interest

Variants as classified as a *Variant of Interest (VOI)* if there is some preliminary/unconfirmed evidence
that the variant possesses genetic markers indicative of an increased rate of transmission, greater disease severity, more detrimental effect on host immunity, and/or reduced likelihood of detection using standard diagnostic assays. A VOI could also be a new variant that has been connected to a large outbreak.

ECDC considers variants to be VOI if there is a possibly of it *“…realistically having an impact on the epidemiological situation in the EU/EEA…”*.

Note that in some countries (e.g. the UK) VOIs may instead be called 'variants under investigation' (VUIs).

##### Variant of Monitoring

A *Variant under Monitoring* (*VUM*) may exhibit some similarities to known VOCs. Data about these variants could come from epidemiological monitoring, screening or preliminary research data.

Increased knowledge about SARS-CoV-2 variants is undoubtedly important for detecting and monitoring the spread of COVID-19 disease and local outbreaks. For example, research into current SARS-CoV-2 variants could facilitate the earlier detection of future variants, and a greater understanding of national and regional trends in spread. Advancing knowledge on SARS-CoV-2 variants is also key in enabling the research community to advise policymakers and governments in planning measures to reduce disease spead, and healthcare resource allocations. This could include, for example, recommendations on when to employ different non-pharmaceutical interventions (NPIs), such as personal protective measures, environmental measures, physical distancing, and travel restrictions.

Over the last two years the research community has advanced our understanding of the SARS-CoV-2 virus at an unprecedented rate. In continuing this work, both now and after the pandemic, the research community contributes to pandemic preparedness; enabling an even faster response in furture pandemics.

For full definitions and table of current **VOCs**, **VOIs**, and **VUMs** see [here](https://www.ecdc.europa.eu/en/covid-19/variants-concern).

**Information sources:** The information on this webpage is primarily based on available information from ECDC, CDC and Swedish Health Agency, as well as recent publications.

## SARS-CoV-2 Variants of Concern

**Table 1.**  Original SARS-CoV-2 variant, first VOC declared, and VOCs currently in Sweden.

<div class="table-responsive">
  <table class="table table-hover table-bordered">
    <thead class="thead-light">
      <tr>
        <th scope="col">Status</th>
        <th scope="col">WHO label</th>
        <th scope="col">Nextstrain</th>
        <th scope="col">Pango lineage</th>
        <th scope="col">Location and date of first detection</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>Original Variant</td>
        <td>19A</td>
        <td>Wuhan/Hu-1/</td>
        <td>-</td>
        <td>China, Dec 2019</td>
      </tr>
      <tr>
        <td>Original VOC</td>
        <td>Alpha</td>
        <td>20I/501Y.V1</td>
        <td>B.1.1.7</td>
        <td>United Kingdom, Sept 2020</td>
      </tr>
      <tr>
        <td>Current VOC</td>
        <td>Beta</td>
        <td>20H/501Y.V2</td>
        <td>B.1.351</td>
        <td>South Africa, Sept 2020</td>
      </tr>
      <tr>
        <td>Current VOC</td>
        <td>Gamma</td>
        <td>20J/501Y.V3</td>
        <td>P.1.</td>
        <td>Brazil, Dec 2020</td>
      </tr>
      <tr>
        <td>Current VOC</td>
        <td>Delta</td>
        <td>21AS/478K</td>
        <td>B.1.617.2</td>
        <td>India, February 2021</td>
      </tr>
      <tr>
        <td>Current VOC</td>
        <td>Omicron</td>
        <td>21K, 21L, 21M</td>
        <td>B.1.1.529</td>
        <td>South Africa and Botswana, Nov 2021</td>
      </tr>
    </tbody>
  </table>
</div>

*Pango lineage refers to [classifications found here](https://cov-lineages.org/). The list of VOCs will be updated by the portal team ASAP after new data becomes available. It was last updated 2021-12-10<!-- can we automate this date?-->. Please see below for more information about the variants listed.*

##### 19A Wuhan/Hu-1/2019

* This is the original SARS CoV-2 variant, which originated in Wuhan, China.
* The first sequences were added to GISAID 26th December 2019 by the National Institute for Communicable Disease Control and Prevention (ICDC), Chinese Center for Disease Control and Prevention, China.

##### Alpha 20I/501Y.V1 (lineage B.1.1.7, VOC202012/01)

* This variant originted in the United Kingdom in September 2020, and has since been referred to as 20I/501Y.V1, VOC 202012/01, B.1.1.7, or Alpha.
* It has a large number of mutations, including spike protein substitutions: 69del, 70del, 144del, (E484K*) (S494P), N501Y, A570D, D614G, P681H, T716I, S982A, D1118H (K1191N*).
* The key mutations in B.1.1.7 are found in the spike protein, there are eight mutations in the spike area. As three spike proteins form a spike on the SARS-CoV-2, all the mutations are found in three positions. A number of other mutations are also found throughout the genome. The spike mutations of most interest are N501Y, D614G, P681H.
* The first cases were detected in Sweden at the end of December 2020.
* This variant had, according to the **Swedish Public Health Agency**, become the dominant strain in most Swedish counties by April 2021. However, as of Dec 2021, the incidence has been drastically reduced by the spread of other variants.
* There is evidence for an impact on *transmissibility*, and *disease severity*, for example [Davies et al Science](https://doi.org/10.1126/science.abg3055), [Davies et al Nature, 2021](10.1038/s41586-021-03426-1) [Shen et al. Cell Host & Microbe, 2021](10.1016/j.chom.2021.03.002).

##### Beta 20H/501Y.V2 (lineage B.1.351)

* This variant originated in South Africa in October 2020. It has also been known as 20H/501Y.V2 or B.1.351.
* Spike protein substitutions present: D80A, D215G, 241del, 242del, 243del, K417N, E484K, N501Y, D614G, A701V. A.
* Spike mutations of interest: K417N, E484K, N501Y, D614G, A701V.
* The first cases were detected in Sweden at the end of December 2020.
* There is evidence for an impact on *transmissibility* [Tegally et al Nature 2021](https://doi.org/10.1038/s41586-021-03402-9), *host immunity* [Cele et al Nature 2021](https://doi.org/10.1038/s41586-021-03471-w), and *disease severity* [Funk et al Eurosurveillance 2021](https://doi.org/10.2807/1560-7917.ES.2021.26.16.2100348).

##### Gamma 20H/501Y.V3 (lineage P.1)

* This variant originated in Brazil. It was first identified in travellers that arrived from Brazil into Japan. The travellers were tested in Tokyo, Japan, and the results were reported by *the National Institute of Infectious Diseases* (NIID), Japan.
* The spike mutations of interest are: K417T, E484K, N501Y, D614G, H655Y.
* This variant was detected in Sweden in February 2021.
* Recent research indicates that the P.1 variant may overcome immunity gained following infection by other SARS CoV-2 variants, and that vaccines may less effective against this variant [Wang P, Wang M, Yu J, et al. 2021, pre-print](https://doi.org/10.1038/s41586-021-03398-2).
* There is evidence for an impact on *transmissibility* [Faria et al. 2021](https://doi.org/10.1126/science.abh2644), *host immunity* [Dejnirattisai et al Cell 2021](https://doi.org/10.1016/j.cell.2021.03.055), and *disease severity* [Funk et al NEJM 2021](https://doi.org/10.2807/1560-7917.ES.2021.26.16.2100348).

##### Delta (lineage B.1.617.2)

* The variant originated in India. It was first detected in February 2021, and was named in May 2021.
* *Delta* is currently (Dec 2021) the most widely spread SARS-CoV-2 variant in Sweden and the world.
* Spike Protein Substitutions: T19R, (V70F*), T95I, G142D, E156-, F157-, R158G, (A222V*), (W258L*), (K417N*), L452R, T478K, D614G, P681R, D950N.
* Spike mutations of interest: L452R, T478K, D614G, P681R.
* This variant was detected in Sweden in April 2021.
* There is evidence of impact on *transmissibility* [UK government](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/988619/Variants_of_Concern_VOC_Technical_Briefing_12_England.pdf.), *disease severity*, [Havervall et al. 2021](https.//doi.org/10.1016/j.ebiom.2021.103523), and *host immunity* [Bernal et al. 2021](https://doi.org/10.1056/NEJMoa2108891).

##### Omicron 21K (lineage B.1.1.629) Current source of outbreak

* The variant was first detected in South Africa and Botswana in November 2021. As of December 2021, it has been detected in over 50 countries, 23 of which are in the EU/EEA.
* [WHO](https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529-sars-cov-2-variant-of-concern)) declared Omicron to be a Variant of Concern (VOC), on 26th November 2021.
* The variant is characterised by 30 changes, including three small deletions and one small insertion in the spike protein. Fifteen of the changes are found in the receptor binding domain. The changes include:
A67V, Δ69-70, T95I, G142D, Δ143-145, Δ211, ins214EPE, G339D, S371L, S373P, S375F, K417N, N440K, G446S, S477N, T478K, E484A, Q493R, G496S, Q498R, N501Y, Y505H, T547K, D614G, H655Y, N679K, P681H, N764K, D796Y, N856K, Q954H, N969K, L981F [ECDC, 2021](https://www.ecdc.europa.eu/sites/default/files/documents/threat-assessment-covid-19-emergence-sars-cov-2-variant-omicron-december-2021.pdf).
* Currently, three subvariants of SATS-CoV-2 Omicron VoC are described: BA.1, BA.2 and BA.3. The most common one worldwide is BA.1, it accounts for most of Omicron VoC cases. However, Scandinavia BA.2 is spreading fast at this point ([Statens Serum Institute, 2022-01-20](https://en.ssi.dk/news/news/2022/omicron-variant-ba2-accounts-for-almost-half-of-all-danish-omicron-cases).
* This variant was first detected in Sweden in Dec 2021.
* Few publications are available to date (less than 25 in PubMed and around 20 in the largest pre-print servers (as of 2021-12-13)).
* There is evidence of an impact on *host immunity* [Pulliam et al. 2021](https://doi.org/10.1101/2021.11.11.212660689). Further, a preliminary result by ([Cele et al](https://www.medrxiv.org/content/10.1101/2021.12.08.21267417v1)) showed a reduction in the neutralisation capacity of antibodies resulting from vaccination compated to other SARS CoV-2 variants.
* Preliminary results from a Swedish cohort study comprising of random blood donors (N=17) state that the mean fold-reduction was around 7, indicating that Omicron has an increased capacity for neutralisation compared to the Delta variant. This infromation was presented on Twitter by [Ben Murrell (@BenjMurrell, 2021-12-07)](https://twitter.com/BenjMurrell/status/1468341478363746310), was referenced in [Nature News](https://www.nature.com/articles/d41586-021-03672-3), and made available as a pre-print [Steward et al. 2021](https://drive.google.com/file/d/1CuxmNYj5cpIuxWXhjjVmuDqntxXwlfXQ/view).
* We provide more details in the [Omicron tab of this page](/omicron).
* For more information about SARS-CoV-2 Omicron variant, see [this Italian Covid-19 Data Portal highlight (2021-12-02)](https://www.covid19dataportal.it/highlights/highlight34/).

## Exploring the variants present in Sweden in detail

#### Nextstrain as a tool

[Nextstrain](https://nextstrain.org) was developed as a tool to enable the tracking of pathogen evolution in real time. It has been used to visualise and explore the relationship between, and transmission of, multiple different viruses (e.g. avian influenza and zika). Users are able to follow simple [step-by-step instructions](https://docs.nextstrain.org/en/latest/index.html) to produce their own 'builds' using genetic sequences. In these 'builds', genetic sequences are used to complete various calculations, and visualisations are produced to display the results.

It is perhaps unsurprising then, that [Nextstrain](https://nextstrain.org) was quickly adopted on a large scale for research related to the SARS-COV-2 pandemic. Since early 2020, it has been used the world over to explore mutations across different SARS-COV-2 variants, visualise the spread of different variants, and to identify potential outbreaks, among other things. In order to expand the data available for use with Nextstrain, and thus the reliability and usefulness of conclusions that could be drawn from these 'builds', it was quickly integrated with data from [GISAID](https://www.gisaid.org). GISAID is a database of genetic sequences for influenza viruses and SARS-COV-2. Public health officials and researchers frequently submit ad make use of the genetic sequence data stored on GISAID. Though the data stored is not truly open, researchers can gain access by applying and agreeing to the conditions set on using the data.

More recently, Nextstrain has begun to produce builds using data from the [European Nucleotide Archive (ENA)](https://www.ebi.ac.uk/ena/browser/home). Data from ENA is completely open, but far fewer sequences are included in this database.

#### Exploring a Nextstrain build for Sweden

<figure class="figure float-right mx-3 w-50">
  <img src="/voc/nextstrain_build.png" alt="Screenshot from the Nextstrain build for Sweden" class="img-thumbnail">
  <figcaption class="figure-caption mt-1"><i>Screenshot from the Nextstrain build for Sweden</i></figcaption>
</figure>

[This Nextstrain build](https://nextstrain.org/groups/neherlab/ncov/sweden), maintained by the Neher lab and Nextstrain team, focusses specifically on sequences from Sweden. There are 4 main visualisation panels:

1. A phylogenetic tree showing the relationships between variants.
2. A geographic map that, by defailt, shows which variants have been detected in Sweden and their relative prevalence. Other aspects of the data related to sequences (e.g. the sampling date) can be displayed instead by using the **Color By** dropdown menu (located to the left of the visualisations panel). **NOTE:** Some data are visible for other countries in the map. This is expected because a subset of sequences from all over the world is needed to properly root the phylogenetic tree (see [here](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/data-prep.html) for information on subsampling procedures).
3. A 'diversity' plot that shows differences in amino acids and nucleotides across variants.
4. A frequency plot showing the relative dominance of variants.

For more information on what is shown in the panels, please see e.g. this [2018 paper by Hadfield et al.](https://academic.oup.com/bioinformatics/article/34/23/4121/5001388), or [this section on interpreting analyses](https://docs.nextstrain.org/en/latest/learn/interpret/index.html) from the Nextstrain documentation. Further, to understand how different visualisations relate to each other, see [the narrative written by Hodcroft et al. in 2020](https://nextstrain.org/narratives/trees-background).

On the left hand side of the visualisations, there is a vertical menu that can be used to animate and change the layout of the visualisations, filter data, and change the view of the tree. See below for infomration on the use of these options

##### Date Range

The **Date Range** option can be used to filter the data so that a specific timeframe is displayed. You can press the 'play' and 'reset' buttons below it to animate the visualisations to play through the period selected. This can be useful if, for example, you want to examine the spread of a new VOC.

##### Color By

The **Color By** menu can be used to select what aspect of the data related to sequences is displayed. You can select e.g. 'clade' to see the variants, 'PANGO Lineage by GISAID', and 'date' (please note that continuous variables like date show as averages, so must be interpreted with this in mind).

##### Tree Options

**Tree Options** can be used to determine how the tree is displayed. You can use the 'Layout' options to select the format in which the tree is displayed, and the 'Branch Length' options to set what determines relative branch length.

##### Map Options

**Map Options** can be used to set the granularity of data displayed on the geographic map, though the options available will depend on the granularity of the data available in GISAID. In each build, 'Region' shows data grouped by continent, 'Country' shows data grouped by country, 'division' typically shows data displayed by county, and 'location' shows data for grouped by smaller subdivisions. In this build, 'Region' primarily shows data for Europe, 'Country' focuses on data from Sweden, 'division' mostly includes data for Swedish counties and some other regions, and 'location' shows data for some smaller subdivisions, though such data is not widely available for Sweden.

##### Other options

The remaining options on the left hand menu, e.g. **Frequency/Animation/Panel Options** are used for changing data display, animation features and toggling the visualisations shown.

## Useful links for further reading about SARS-CoV-2 variants

This section includes links to other sources of information about variants that can be used to find out more. Note that links from all over the world are included, not just Sweden.

##### Swedish Health Agency (Swedish Folkhälsomyndigheten)
Detection and statistics of *variants of concern* by *Swedish Health Agency* are found [here](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/statistik-och-analyser/sars-cov-2-virusvarianter-av-sarskild-betydelse/) and [here](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/statistik-och-analyser/sars-cov-2-virusvarianter-av-sarskild-betydelse/sars-cov-2-virusvarianter-av-sarskild-betydelse/).
*Please note that these pages are only available in Swedish*.

##### ECDC
[ECDC Dashboard](https://www.ecdc.europa.eu/en/covid-19/situation-updates/variants-dashboard) shows a dashboard of SARS-CoV-2 variants in EU/EEA.
[ECDC country overview report](https://covid19-country-overviews.ecdc.europa.eu/#4_Variants_of_concern)
[Data on SARS-CoV-2 variants in the EU/EEA](https://www.ecdc.europa.eu/en/publications-data/data-virus-variants-covid-19-eueea). *Note that the data shown in the EDCD Dashboard are sourced from the European Surveillance System [TESSy]( https://www.ecdc.europa.eu/en/publications-data/european-surveillance-system-tessy  ) or from the [GISAID EpiCoV database](https://www.gisaid.org).*

##### Centre for Disease Control (CDC)
Information about SAS-CoV-2 variants is found [here](https://www.cdc.gov/coronavirus/2019-ncov/variants/about-variants.html)

##### Danish Health Authority
Information about Covid-19 statistics and charts are found [here](https://www.sst.dk/en/english/corona-eng/status-of-the-epidemic/covid-19-updates-statistics-and-charts )

##### Finnish Institute for Health and Welfare
Information about Covid-19 variants is found [here](https://thl.fi/en/web/infectious-diseases-and-vaccinations/what-s-new/coronavirus-covid-19-latest-updates/transmission-and-protection-coronavirus/coronavirus-variants#omicron)

##### Links from the UK
Information about variants circulating in the UK (typically reflective of those in Sweden) can be found dashboards produced by [Wellcome Sanger](https://covid19.sanger.ac.uk/lineages/raw?lineage=B.1.1.529) and [COG-UK](https://sars2.cvr.gla.ac.uk/cog-uk/). The UK government also regularly [publishes information](https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers) about the numbers of cases attributed to different variants.

<hr>

**References:** The information on this webpage is based on information from primarily ECDC, CDC, Swedish Public Health Agency, as well as publications.
