---
title: SARS-CoV-2 Variants of Concern (VoC)
toc: true
---

<div class="row mb-4"><div class="col col-sm-8"><ul class="nav nav-tabs nav-justified">
  <li class="nav-item">
    <a class="nav-link active" href="#"><b>Variants of Concern (VoC)</b></a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="omicron">Omicron VoC</a>
  </li>
</ul></div></div>

## Background and nomenclature

SARS-CoV-2 is part of the group of *RNA viruses*. The RNA viruses are known for fast mutation rates and for resilience to antiviral drugs. A virus variant can have one, or more, mutations that differentiate it from other virus variants circulating in a population.
Since the start of the COVID-19 pandemic in December 2020 a number of variants have emerged in different part of the world. While the research community and governmental health agencies are monitoring all variants that have emerged, some variants are given more attention and monitored more closely due to their attributes and characteristics. The research community has used [Pango](https://cov-lineages.org), [Nextstrain]( https://nextstrain.org ) and [GISAID](https://www.gisaid.org ) nomenclature to distinguish these new variant. However, in May 2021, the World Health Organization (WHO) proposed to use [labels from the Greek alphabet for naming SARS CoV-2 variants](https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/=) to be used, alongside the scientific nomenclature, when communicating for the public. To-date Omicron is the latest variant designated a Greek letter.

[*The European Centre for Disease Control*](https://www.ecdc.europa.eu/en ) are dividing the SARS-CoV-2 virus variants into:

* *Variant of Concern* (*VOC*)
* *Variant of Interest* (*VOI*)
* *Variant under monitoring* (*VUM*)

##### Variant of concern

A *Variant of concern* (*VOC*) have clear information that indicates that the variant will *impact how fast the variant can be transmitted, severity of the disease, and/or immunity, , which can have effect on the epidemiological situation in a society.* Available research about the *VOC* could also be indicating *vaccine escape* (significant reduced neutralizing effect by antibodies generated from vaccination), or decreased vaccine protection from severe disease. In addition, a SARS-CoV-2 *VOC* could also have proved hard to detect using standard diagnostic assays that has previously been used for other variants and therefore more difficult to track. In current *VOCs* in Sweden are: **Beta**, **Gamma**, **Delta**, and **Omicron**.

##### Variant of Interest

For a *Variant of Interest* (*VOI*) there is evidence, though preliminary or not yet confirmed, indicating that for example these variants could have genetic markers suggested or predicted to affect *transmission*, evidence of impact on *severity* and /or *immunity*, or *diagnostics*. A variant of interest could also be a new variant that has been connected to a large outbreak.
ECDC also adds that a VOI should *“…realistically having an impact on the epidemiological situation in the EU/EEA…”*

##### Variant of Monitoring (VOM)

A *Variant under monitoring* (*VOM*) may have properties that could indicate similarities to known VOCs. The data about these could come from epidemiological monitoring, screening or preliminary research data.

In summary, increased knowledge about *SARS-CoV-2 variants* is important for detecting and monitoring spread of the disease and local outbreaks. Research into present-day SARS-CoV-2 variants may contribute to earlier detection of new variants, understanding national and regional trends in spread, as well as comparing virus variants. In addition, the research community may also contribute important real-time data to support policymakers and government in healthcare planning, development of i.e., for non- pharmaceutical interventions (NPIs) (such as i.e., personal protective measures, environmental measures, physical distancing, and travel related measures), and other measures to mitigate the spread of SARS-CoV-2. Over the last two years the research community has, in record time, increase the knowledge and understanding about the new SARS-CoV-2 virus, and by continuing this work contributes to societies future pandemic preparedness.

For full definitions and table of current **VOCs**, **VOIs**, and **VOMs**  see [here](https://www.ecdc.europa.eu/en/covid-19/variants-concern). *Note: The information on this webpage is primarily based on available information from  ECDC, CDC.and Swedish Health Agency, as well as recent publications*

## SARS-CoV-2 Variants of Concern

**Table 1.**  SARS-CoV-2 original strain, first VOC  and four VOCs currently monitored in Sweden: *beta. gamma, delta and omicron*.

| WHO label      | Nextstrain | Pango lineage| First detected and when |
| ----------- | ----------- |------- |--------- |
| 19A      | Wuhan/Hu-1/    |  -      | China, Dec 2019 |
| Alpha   | 20I/501Y.V1       | B.1.1.7	|  United Kingdom, Sept 2020|
| Beta  | 20H/501Y.V2 | B.1.351 | South Africa, Sept 2020 |
| Gamma | 20J/501Y.V3 | P.1.|Brazil, Dec 2020|
| Delta |21AS/478K |B.1.617.2 |India, Dec 2021 |
| Omicron |21K| B.1.1.529 | South Africa and Botswana, Nov 2021 |

*Pango lineage refers to https://cov-lineages.org/*
*The list of VOCs is updated when new knowledge is added, today (2021-12-10)**

**19A	Wuhan/Hu-1/2019**

* This is the original SARS CoV-2 variant, it originated from Wuhan, China.
* The first sequences were added Dec 26 2020 to GISAID by National Institute for Communicable Disease Control and Prevention (ICDC) Chinese Center for Disease Control and Prevention, China.

**Alpha  20I/501Y.V1**	 (lineage B.1.1.7, VOC202012/01)
* This variant originated in the United Kingdom in Sept 2020, it is known as known as 20I/501Y.V1, VOC 202012/01, or B.1.1.7) and has a large number of mutations.
* Spike Protein Substitutions: 69del, 70del, 144del, (E484K*), (S494P*), N501Y, A570D, D614G, P681H, T716I, S982A, D1118H (K1191N*). The key mutations in B.1.1.7 are found in the spike protein, there are eight mutations in the spike area. As three spike proteins form a spike on the SARS-CoV-2 all the mutation are found in three positions. A number of other mutations are also found throughout the genome. The spike mutations of interest are N501Y, D614G, P681H.
* The first cases were detected in Sweden in the end of December 2020.
* The variant B.1.1.7 has according the **Swedish Public Health Agency** became the predominant strain in most counties (April 2021) but today Dec 2021 is has been drastically decreased since Delta started to spread.
* Evidence for impact on *transmissibility*, and *severity* for example Wang et al. MedRxiv 2020, [Davies et al Science](https://doi.org/10.1126/science.abg3055) [Davies et al Nature, 2021](10.1038/s41586-021-03426-1) [Shen et al. Cell Host & Microbe, 2021](10.1016/j.chom.2021.03.002)

**Beta 20H/501Y.V2**	 (lineage B.1.351)

* This variant originated in South Africa, it is also known as 20H/501Y.V2 or B.1.351)
* Spike Protein Substitutions: D80A, D215G, 241del, 242del, 243del, K417N, E484K, N501Y, D614G, A701V. A
* Spike mutations of interest: K417N, E484K, N501Y, D614G, A701V
* The first cases were detected in Sweden in the end of December 2020.
* Evidence for impact on *transmissibility* (Tegally et al Nature 2021](https://doi.org/10.1038/s41586-021-03402-9), *immunity*  [Cele et al Nature 2021](https://doi.org/10.1038/s41586-021-03471-w), and *severity* Funk et al Eurosurveillance 2021](https://doi.org/10.2807/1560-7917.ES.2021.26.16.2100348)

**Gamma	20H/501Y.V3** (lineage P.1))

* The variant originated in Brazil. It was first was identified in travelers from Brazil, who were tested Tokyo, Japan and reported by *the National Institute of Infectious Diseases* (NIID) of Japan.
* The spike mutations of interest are: K417T, E484K, N501Y, D614G, H655Y
* This variant was detected in Sweden in February 2021.
* Recent research indicate that the P.1 variant may overcome immunity from other SARS CoV-2 infections and vaccines may not work as well against P1 variant. Wang P, Wang M, Yu J, et al. 2021, preprint).
* Evidence for impact on *transmissibility*  [Faria et al. 2021](https://doi.org/10.1126/science.abh2644
), *immunity*, [Dejnirattisai et al Cell 2021](https://doi.org/10.1016/j.cell.2021.03.055), and *severity*  [Funk et al NEJM 2021](https://doi.org/10.2807/1560-7917.ES.2021.26.16.2100348))

**Delta	(lineage 6.1.617.2)**

* The variant originated in India and was first detected in February 2021, it was named in May 2021.
* *Delta* is currently the most widely spread SARS-CoV-2 variant in Sweden and in the world (Dec 2021).
* Spike Protein Substitutions: T19R, (V70F*), T95I, G142D, E156-, F157-, R158G, (A222V*), (W258L*), (K417N*), L452R, T478K, D614G, P681R, D950N.
* Spike mutations of interest: L452R, T478K, D614G, P681R
* This variant was detected in Sweden in April 2021.
* Evidence of impact on *transmissibility* [UK government](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/988619/Variants_of_Concern_VOC_Technical_Briefing_12_England.pdf.), *severity*, [Havervall et al. 2021](https.//doi.org/10.1016/j.ebiom.2021.103523), and *immunity* [Bernal et al. 2021](https://doi.org/10.1056/NEJMoa2108891).

## Exploring relationships between variants present in Sweden using Nextstrain

#### Nextstrain as a tool

[Nextstrain](https://nextstrain.org) was developed as a tool to enable the tracking of pathogen evolution in real time. It has been used to visualise and explore the relationship between and transmission of multiple different viruses (e.g. avian influenza and zika). Users could make use of the tool by uploading genetic sequences and following simple [step-by-step instructions](https://docs.nextstrain.org/en/latest/index.html) to produce their own 'builds'. In these 'builds' the desired calculations are completed using the sequences, and visualisations of the results are produced that can be viewed by the user.

It is perhaps unsurprising then, that [NextStrain](https://nextstrain.org) was quickly adopted on a large scale for research related to the SARS-COV-2 pandemic. Since early 2020, it has been used the world over to explore mutations across different SARS-COV-2 variants, visualise the spread of different variants, and to identify potential outbreaks, among other things. In order to expand the data available for use with NextStrain, and thus the reliability and usefulness of conclusions that could be drawn using the tool, it was quickly intergrated with data from [GISAID](https://www.gisaid.org). GISAID is a database containing genetic sequences for influenza visuses and SARS-COV-2. It is used widely by reearchers and public health authorities, who also frequently submit their own data for use. Though the data stored is not truly open, researchers can gain access by applying and agreeing to the conditions set on using the data.

#### Exploring a NextStrain build for Sweden

[This NextStrain build](https://nextstrain.org/groups/neherlab/ncov/sweden), maintained by the Neher lab and NextStrain team, focusses specifically on sequences from Sweden. There are 4 main visualisation panels:

1. A phylogenetic tree showing the relationships between variants.
2. A geographic map showing which variants have been detected in Sweden and their relative prevalence by default. Other aspects of the data related to sequences (e.g. the sampling date) can be displayed instead using the **Color By** dropdown menu to the left of the visualisations panel. **NOTE:** Some data are visible for other countries, as a sample from all over the world is needed to properly root the phylogenetic tree (see [here](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/data-prep.html) for information on subsampling procedures).
3. A 'diversity' plot that shows information related to differences in amino acids and nucleotides across variants.
4. A frequency plot showing the relative dominance of variants.

For more information on what is shown in the panels, please see e.g. this [2018 paper by Hadfield et al.](https://academic.oup.com/bioinformatics/article/34/23/4121/5001388), or [this section on interpreting analyses](https://docs.nextstrain.org/en/latest/learn/interpret/index.html) in the Nextrain documentation. Further, to understand how different visuaalisations relate to each other, see [the narrative written by Hodcroft et al. in 2020](https://nextstrain.org/narratives/trees-background).

On the left hand side of the visualisations, there is a vertical menu that can be used to animate and change the layout of the visulations, filter data, and change the view of the tree. For example, you can use the **Date Range** option to filter the data so that a specific timeframe is displayed. You can press the 'play' and 'reset' buttons below it to animate the visualisations to play through the period selected. The **Color By** menu can be used to select what aspect of the data related to sequences is displayed. You can select e.g. 'clade' to see the variants, 'PANGO Lineage by GISAID', and 'date' (note that continuous variables like date show as averages). **Tree Options** can be used to determine how the tree is displayed. You can use the 'Layout' options to select how the tree is displayed, andthe 'Branch Length' options to set what determines relative branch length. **Map Options** can be used to set the granularity of data displayed on the geographic map (options depend on the granularity available in GISAID); 'Region' shows data grouped by continent, 'Country' shows data grouped by country, 'division' typically shows data displayed by county, and 'location'shows data for grouped by smaller subdivisions (not widely availabel for Sweden). The remaining options on the left hand menu, e.g. **Frequency/Animation/Panel Options** are used for changing data display, animation features and toggling the visualisations shown.

##  Useful links for further reading about SARS-CoV-2 variants

**Swedish Health Agency (Swedish Folkhälsomyndigheten)**
Detection and statistics of *variants of concern* by *Swedish Health Agency* are found [here](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/statistik-och-analyser/sars-cov-2-virusvarianter-av-sarskild-betydelse/) and [here](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/statistik-och-analyser/sars-cov-2-virusvarianter-av-sarskild-betydelse/sars-cov-2-virusvarianter-av-sarskild-betydelse/).
*Please note that these pages are only available in Swedish*.

**ECDC** [ECDC Dashboard](https://www.ecdc.europa.eu/en/covid-19/situation-updates/variants-dashboard) shows a dashboard of SARS-CoV-2 variants in EU/EEA.
[ECDC country overview report](https://covid19-country-overviews.ecdc.europa.eu/#4_Variants_of_concern)
[Data on SARS-CoV-2 variants in the EU/EEA](https://www.ecdc.europa.eu/en/publications-data/data-virus-variants-covid-19-eueea). *Note that the data shown in the EDCD Dashboard are sourced from the European Surveillance System [TESSy]( https://www.ecdc.europa.eu/en/publications-data/european-surveillance-system-tessy  ) or from the [GISAID EpiCoV database](https://www.gisaid.org).*

**Centre for Disease Control (CDC)** information about SAS-CoV-2 variants is found [here](
https://www.cdc.gov/coronavirus/2019-ncov/variants/about-variants.html)

**Danish Health Authority** information about Covid-19 statistics and charts are found [here](
https://www.sst.dk/en/english/corona-eng/status-of-the-epidemic/covid-19-updates-statistics-and-charts )

**Finish Institute for Health and Welfare** information about Covid-19 variants is found [here](https://thl.fi/en/web/infectious-diseases-and-vaccinations/what-s-new/coronavirus-covid-19-latest-updates/transmission-and-protection-coronavirus/coronavirus-variants#omicron)

**Sequencing and genomic information using Nextstrain:**

- https://nextstrain.org/groups/neherlab/ncov/sweden
- https://nextstrain.org/sars-cov-2/
- https://nextstrain.org/ncov/gisaid/global
