---
title: "Omicron variant of concern: ongoing research, data, publications and preprints"
toc: true
aliases:
    - /omicron/
---

<div class="row mb-4"><div class="col col-sm-8"><ul class="nav nav-tabs nav-justified">
  <li class="nav-item">
    <a class="nav-link" href="../">Variants of Concern (VOC)</a>
  </li>
  <li class="nav-item">
    <a class="nav-link active" href="#"><b>Omicron<br></b></a>
  </li>
</ul></div></div>

On this page we have compiled current knowledge on Omicron, the latest SARS-CoV-2 variant to be designated as a Variant of Concern (VOC). This includes availble data, ongoing research efforts, initial findings, among other things related to Omicron. Please see [this page for information on other VOCs](../).

We make all possible efforts to track preprints, publications, and other resources related to Omicron and link them below. Researchers and organisations from around Sweden are also welcome to submit their datasets and to make relevant announcements on this page. If you feel like something is missing that should be included, or you wish to make a submission/announcement, please use the forms below or email us at datacentre@scilifelab.se. The team behind the COVID-19 Data Portal continuously monitors incoming suggestions, and new items are published as soon as possible.

## Available data

This section presents a list of the available data related to Omicron (or related work that could benefit research into this variant) shared by researchers affiliated to a Swedish research institute. We welcome the submission of suggestions for additional datasets, either using the form below or by e-mail: datacentre@scilifelab.se. All submissions will be moderated by the Portal team before they appear on the page.

{{< omicron_page_suggest_available_data >}}

#### Data available from research groups in Sweden

{{< omicron_page_av_data_swe >}}

#### Data available from research groups in other countries

{{< omicron_page_av_data_int >}}

## Ongoing research efforts

In this section, we present announcements about ongoing research efforts from researchers and organisations in Sweden. Please feel free to share what you are currently working on and/or your results, to seek collaborators to aid with aspects of your work (e.g. data, equipment, reagents, etc.). Any announcement useful for advacing research on Omicron is welcome. Feel free to submit the information either by using the form below or by sending us an e-mail to datacentre@scilifelab.se.

{{< omicron_page_add_announcement >}}

{{< omicron_page_announcements >}}

## Exploring Omicron VoC in Sweden using Nextstrain

<figure class="figure float-right mx-3 w-50">
  <img src="/voc/nextstrain_omicron.png" alt="Screenshot from the Nextstrain build" class="img-thumbnail">
  <figcaption class="figure-caption mt-1"><i>Screenshot from the Nextstrain build</i></figcaption>
</figure>

There is a [Nextstrain build maintained by the Neher lab and NextStrain team](https://nextstrain.org/groups/neherlab/ncov/sweden) that focusses specifically on sequences from Sweden, follow [this link for Omicron focus](https://nextstrain.org/groups/neherlab/ncov/sweden?f_clade_membership=21K%20%28Omicron%29). Please keep in mind that quality of the analyses by Nextstrain depends on the amount and quality of available data so the results should be taken with caution.

There are 4 main visualisation panels in Nextstrain:

- A phylogenetic tree showing the relationships between variants.
- A geographic map showing which variants have been detected in Sweden and their relative prevalence by default. Other aspects of the data related to sequences (e.g. the sampling date) can be displayed instead using the **Color By** dropdown menu to the left of the visualisations panel. **NOTE:** Some data are visible for other countries, as a sample from all over the world is needed to properly root the phylogenetic tree (see [here](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/data-prep.html) for information on subsampling procedures).
- A 'diversity' plot that shows information related to differences in amino acids and nucleotides across variants.
- A frequency plot showing the relative dominance of variants.

For more information on what is shown in the panels, please see e.g. this [2018 paper by Hadfield et al.](https://academic.oup.com/bioinformatics/article/34/23/4121/5001388), or [this section on interpreting analyses](https://docs.nextstrain.org/en/latest/learn/interpret/index.html) in the Nextrain documentation. Further, to understand how different visuaalisations relate to each other, see [the narrative written by Hodcroft et al. in 2020](https://nextstrain.org/narratives/trees-background).

## Relevant publications and preprints

Here we display a selection of preprints and journal publications reporting results on Omicron VoC.

#### Publications and preprints with participation of research groups in Sweden

{{< omicron_page_pubs_swe >}}

#### Publications and preprints by research groups in other countries

{{< omicron_page_pubs >}}

## Other useful resources

Here we list a number of useful resources related to the new *SARS-CoV-2 Omicron variant*.

- [Information from the **WHO** about *Omicon* as a variant of concern (2021-11-26)](https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern)
- [Epidemiological reports from **ECDC**](https://www.ecdc.europa.eu/en/news-events)
- [Data on SARS-CoV-2 variants in the EU/EEA from **ECDC**](https://www.ecdc.europa.eu/en/publications-data/data-virus-variants-covid-19-eueea)
- [Information about *Omicron* from the **CDC**](https://www.cdc.gov/coronavirus/2019-ncov/variants/omicron-variant.html)
- [Information about *Omicron* from the **Statens Serum Institute, Denmark**](https://www.ssi.dk/aktuelt/nyheder/2021/status-pa-omikron-varianten-b11529-pr-061221)
- [Statens Serum Institute, Denmark Rapport Dec 12 2021](https://covid19.ssi.dk/virusvarianter/omikron)
- [Statistics about cases of Omicron from the **Danish Statens Serum Institute**](https://files.ssi.dk/covid19/podepind-sekventering/variant-pcr-test-december2021/opgoerelse-variantpcr-covid19-13122021-j49o)
- [Information about Omicron from the **Norwegian Institute of Public Health**](https://www.fhi.no/en/id/infectious-diseases/coronavirus/statistikk-over-meldte-tilfeller-avny-virusvariant-omikron/)
- [For lineages and information related to sequences from **Cov-lineages**](https://cov-lineages.org/global_report_B.1.1.529.html)
- [Lineages and information related to sequences from **Nextstrain**](https://nextstrain.org/groups/neherlab/ncov/sweden)
- [Assembled information about the Omicron from the **PHA4GE** (Bioinformatics Pipelines & Visualization Working Group)](https://github.com/pha4ge/pipeline-resources/blob/main/docs/omicron-resources.md)
- [**VEO** report on mutations and variation in publicly shared SARS-CoV-2 raw sequencing data, Report No. 10, 15 December 2021](https://www.veo-europe.eu/news/nyhed?id=%7B1B87370A-0EBB-4412-81E5-9390A6FCDC07%7D)
- [Proportion of different variants of concern in positive samples sequenced in **Region Sk√•ne**](https://www.skane.se/digitala-rapporter/lagesbild-covid-19-i-skane/sekvensering/)
- [Ontario COVID-19 Dashboard: Tracking Omicron](https://covid19-sciencetable.ca/ontario-dashboard/)
- [ESCMID Urgent update on Omicron SARS-CoV-2 variant, 2021-12-22](https://eacademy.escmid.org/escmid/2021/covid-19-webinars/348220/)
- [Statens Serum Institut: Antigen tests can detect new variants, 2022-01-07](https://www.ssi.dk/aktuelt/nyheder/2022/antigentest-undersoger-for-varianter)
- [Nextstrain: Phylogenetic analysis of SARS-CoV-2 clusters in their international context - cluster DanishCluster](https://nextstrain.org/groups/neherlab/ncov/DanishCluster)

## Get in touch with us

Do you have suggestions or other feedback? Get in touch with us by e-mailing datacentre@scilifelab.se.

{{< omicron_page_form_scripts >}}
