---
title: "Omicron variant of concern: ongoing research, data, publications and preprints"
toc: true
aliases:
    - /omicron/
---

<div class="row mb-4"><div class="col col-sm-8"><ul class="nav nav-tabs nav-justified">
  <li class="nav-item">
    <a class="nav-link" href="../">Variants of Concern (VOC)</a>
  </li>
  <li class="nav-item">
    <a class="nav-link active" href="#"><b>Omicron<br>(Most recent VOC)</b></a>
  </li>
</ul></div></div>

On this page we have compiled current knowledge on Omicron, the latest SARS-CoV-2 variant to be designated as a Variant of Concern (VOC). This includes any information availble data, ongoing research efforts, initial findings, among other things related to Omicron. Please see [this page for information on other VOCs](../).

We make all possible efforts to track pre-prints, publications, and other resources related to Omicron and link them below. Researchers and organisations from around Sweden are also welcome to submit their datasets and to make relevant announcements on this page. If you feel like something is missing that should be included, or you wish to make a submission/announcement, please use the forms below or email us at datacentre@scilifelab.se. The team behind the COVID-19 Data Portal continuously monitors incoming suggestions, and new items are published as soon as possible.

## Available data

This section presents a list of the available data related to Omicron (or related work that could benefit research into this variant) shared by researchers affiliated to a Swedish research institute. We welcome the submission of suggestions for additional datasets, either using the form below or by e-mail: datacentre@scilifelab.se. All submissions will be moderated by the Portal team before they appear on the page.

<!-- Button trigger modal -->
<button type="button" class="btn btn-primary mb-2" data-toggle="modal" data-target="#exampleModalTwo">
  Add available data
</button>

<!-- Modal -->
<div class="modal fade" id="exampleModalTwo" tabindex="-1" role="dialog" aria-labelledby="exampleModalTwoLabel" aria-hidden="true">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title" id="exampleModalTwoLabel">Add available data</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        [form will appear here]
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Submit</button>
      </div>
    </div>
  </div>
</div>

<div mb-2>
<b><a target="_blank" href="https://doi.org/10.6084/m9.figshare.17129771.v1">Supplementary information: Structural-bioinformatics analysis of SARS-CoV-2 variants reveals higher hACE2 receptor binding affinity for Omicron B.1.1.529 spike RBD compared to wild-type reference.</a></b><br>
<span class="text-muted">Durmaz, Vedat; Köchl, Katharina; Singh, Amit; Hetmann, Michael; Parigger, Lena; Krassnigg, Andreas</span><br>
<i>figshare</i> (2021). 10.6084/m9.figshare.17129771.v1.
</div><hr>

<div mb-2>
<b><a target="_blank" href="https://doi.org/10.6084/m9.figshare.17105090.v1">Genomic sequences of Omicron variants of SARS-CoV-2 from November 9 to November 28, 2021.</a></b><br>
<span class="text-muted">Yeh, Ting Yu; Contreras, Gregory</span><br>
<i>figshare</i> (2021). 10.6084/m9.figshare.17105090.v1.
</div><hr>

<div mb-2>
<b><a target="_blank" href="https://www.ecdc.europa.eu/en/covid-19/data">COVID-19 datasets from the European Centre for Disease Prevention and Control.</a></b><br>
<span class="text-muted">European Centre for Disease Prevention and Control</span><br> (2021).
</div><hr>

<p class="text-muted">No openly available data from Swedish researchers tracked as of Dec 14.</p>

## Ongoing research efforts

In this section, we present announcements about ongoing research efforts from researchers and organisations in Sweden. Please feel free to share what you are currently working on and/or your results, to seek collaborators to aid with aspects of your work (e.g. data, equipment, reagents, etc.). Any announcement useful for advacing research on Omicron is welcome. Feel free to submit the information either by using the form below or by sending us an e-mail to datacentre@scilifelab.se.

<!-- Button trigger modal -->
<button type="button" class="btn btn-primary mb-2" data-toggle="modal" data-target="#exampleModal">
  Add an announcement
</button>

<!-- Modal -->
<div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title" id="exampleModalLabel">Add an announcement</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        [form will appear here]
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Submit</button>
      </div>
    </div>
  </div>
</div>

<div class="alert alert-info mb-2">
  <h6>Sequences available: 10 Omicron samples</h5>
  <p class="small mb-2 text-muted">2021-12-14 15:34</p>
  <p>The Hammond lab are now running sequencing of SARS-CoV-2 variant. In our last batch of 100 samples, 10 were found to be the new Omicron. Please find the metadata record in the SciLifeLab Data Repository (doi.xxx) where we describe our work and how other researchers request access to the  bam.files. More sequences are run weekly. We are always interested in establishing new collaborations contact Dr Hammond.</p>
  <p><b>Contact:</b> Lucas Hammond, <a href="#">lucas.hammond@umu.se</a></p>
</div>

<div class="alert alert-info mb-2">
  <h6>Interested in collaborators running MS</h5>
  <p class="small mb-2 text-muted">2021-12-14 15:34</p>
  <p>A have an open call coming up and our group are interested in applying together with other labs with a similar interest in how well vaccinations protects against new variants. We are running the serology but are interested in collaborators running MS. Data Centre has kindly agreed to set interested labs in contact with me. Best regards Maria A.</p>
  <p><b>Contact:</b> Maria A, <a href="#">a.maria@gu.se</a></p>
</div>

<div class="alert alert-info mb-2">
  <h6>Looking for in-house/commercial primers for detection of Omicron with RT-qPCR</h5>
  <p class="small mb-2 text-muted">2021-12-14 12:20</p>
  <p>The Andersson Lab has been working on influenza since 2010. Recent development of the pandemic and the emergence or another SARS-CoV2- variant has made us decide to start working on variant detection. Would you be willing to share “in-house” made/commercial primers for detection of Omicron with RT-qPCR?. We hope to get our work up and running before Christmas. Data Centre can help set us in contact. Thank you!</p>
  <p><b>Contact:</b> Niclas Andersson, <a href="#">niclas.andersson@uu.se</a></p>
</div>

<!-- <div class="card mb-2">
  <div class="card-body">
    <h5 class="card-title">This is the title of the announcement</h5>
    <h6 class="card-subtitle mb-2 text-muted">2021-12-14 15:34</h6>
    <p class="card-text">This is where the text of the announcement goes. The Swedish COVID-19 Data Portal provides information, guidelines, tools and services to support researchers to utilise Swedish and European infrastructures for data sharing. The portal is a national node of the European COVID-19 Data Portal.</p>
    <a href="#" class="card-link">name.surname@uu.se</a>
  </div>
</div> -->

## Relevant publications and preprints

<div mb-2>
<b><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/34873578/">Omicron (B.1.1.529): Infectivity, vaccine breakthrough, and antibody resistance.</a></b><br>
<span class="text-muted">Chen, Jiahui; Wang, Rui; Gilby, Nancy Benovich; Wei, Guo-Wei</span><br>
<i>ArXiv</i> (2021-12-01). PMCID: PMC8647651.
<div class="row">
                        <a class="btn btn-link text-dark" data-toggle="collapse" href="#abstract1"
                          role="button" aria-expanded="false" aria-controls="abstract1">
                          Abstract <i class="fas fa-caret-down"></i></a>
                        <div class="collapse" id="abstract1">
                          <div class="card card-body">The latest severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variant Omicron (B.1.1.529) has ushered panic responses around the world due to its contagious and vaccine escape mutations. The essential infectivity and antibody resistance of the SARS-CoV-2 variant are determined by its mutations on the spike (S) protein receptor-binding domain (RBD). However, a complete experimental evaluation of Omicron might take weeks or even months. Here, we present a comprehensive quantitative analysis of Omicron's infectivity, vaccine-breakthrough, and antibody resistance. An artificial intelligence (AI) model, which has been trained with tens of thousands of experimental data points and extensively validated by experimental data on SARS-CoV-2, reveals that Omicron may be over ten times more contagious than the original virus or about twice as infectious as the Delta variant. Based on 132 three-dimensional (3D) structures of antibody-RBD complexes, we unveil that Omicron may be twice more likely to escape current vaccines than the Delta variant. The Food and Drug Administration (FDA)-approved monoclonal antibodies (mAbs) from Eli Lilly may be seriously compromised. Omicron may also diminish the efficacy of mAbs from Celltrion and Rockefeller University. However, its impact on Regeneron mAb cocktail appears to be mild.</div>
                        </div>
                      </div>
</div><hr>

<div mb-2>
<b><a target="_blank" href="https://doi.org/10.1002/jmv.27503">Detection of highly muted coronavirus variant Omicron (B.1.1.529) is triggering the alarm for South Asian countries: Associated risk factors and preventive actions.</a></b><br>
<span class="text-muted">Daria, Sohel; Bhuiyan, Mohiuddin Ahmed; Islam, Md Rabiul</span><br>
<i>J Med Virol</i> (2021-12-04). DOI: 10.1002/jmv.27503.
</div><hr>

<div mb-2>
<b><a target="_blank" href="https://doi.org/10.1038/d41586-021-03592-2">Omicron is supercharging the COVID vaccine booster debate.</a></b><br>
<span class="text-muted">Dolgin, Elie</span><br>
<i>Nature</i> (2021-12-04) pages. DOI: 10.1038/d41586-021-03592-2.
</div><hr>

<div mb-2>
<b><a target="_blank" href="https://doi.org/10.1101/2021.12.09.21267556">Predicted Symptomatic Effectiveness of Pfizer-BioNTech BNT162b2 Vaccine Against Omicron Variant of SARS-CoV-2</a></b><br>
<span class="text-muted">[Authors]</span><br>
<i>medRxiv</i> (2021-12-11). DOI: 10.1101/2021.12.09.21267556. <span class="preprint-icon">preprint</span>
<div class="row">
                        <a class="btn btn-link text-dark" data-toggle="collapse" href="#abstract2"
                          role="button" aria-expanded="false" aria-controls="abstract2">
                          Abstract <i class="fas fa-caret-down"></i></a>
                        <div class="collapse" id="abstract2">
                          <div class="card card-body">This paper presents predictions of the symptomatic effectiveness of the Pfizer-BioNTech BNT162b2 (Comirnaty) vaccine against Omicron B.1.1.529, the latest SARS-CoV-2 variant of concern. They were obtained assuming fold decreases in Omicron neutralisation by vaccine-induced antibodies versus neutralisation of the virus Wild Type. A 25-fold decrease was assumed based on Omicron pseudovirus neutralisation study by Pfizer and BioNTech; a 94-fold, based on live-Omicron neutralisation study in South Africa; and 40, 80 and 120 folds, hypothesised based on genetic information. The effectiveness of two vaccine doses was predicted as 66% (42, 86), 48% (25, 72) and 42% (20, 66) for up to five months starting 2-4 weeks after the second dose, for the 25, 80 and 120 folds, respectively. The effectiveness of booster vaccination was predicted under a highly conservative assumption that the third dose would increase neutralisation by only 3.3 folds compared to the second dose. The predictions of effectiveness for up to five months, starting 2-4 weeks after the third dose, were 81% (59, 95), 67% (43, 87) and 61% (37, 82) for the 25, 80 and 120 folds, respectively. Despite the large fold decreases considered, the vaccine could still provide substantial protection, particularly after a booster and against severe disease. The paper is accompanied by free software which can be used to predict the symptomatic effectiveness of Comirnaty against Omicron under different neutralisation folds, including those obtained experimentally.</div>
                        </div>
                      </div>
</div><hr>

[....]

## Exploring relationships between variants present in Sweden using Nextstrain

[This Nextstrain build](https://nextstrain.org/groups/neherlab/ncov/sweden), maintained by the Neher lab and NextStrain team, focusses specifically on sequences from Sweden. There are 4 main visualisation panels:

1. A phylogenetic tree showing the relationships between variants.
2. A geographic map showing which variants have been detected in Sweden and their relative prevalence by default. Other aspects of the data related to sequences (e.g. the sampling date) can be displayed instead using the **Color By** dropdown menu to the left of the visualisations panel. **NOTE:** Some data are visible for other countries, as a sample from all over the world is needed to properly root the phylogenetic tree (see [here](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/data-prep.html) for information on subsampling procedures).
3. A 'diversity' plot that shows information related to differences in amino acids and nucleotides across variants.
4. A frequency plot showing the relative dominance of variants.

For more information on what is shown in the panels, please see e.g. this [2018 paper by Hadfield et al.](https://academic.oup.com/bioinformatics/article/34/23/4121/5001388), or [this section on interpreting analyses](https://docs.nextstrain.org/en/latest/learn/interpret/index.html) in the Nextrain documentation. Further, to understand how different visuaalisations relate to each other, see [the narrative written by Hodcroft et al. in 2020](https://nextstrain.org/narratives/trees-background).

## Other useful resources

Here we list a number of useful resources related to the new *SARS-CoV-2 Omicron variant* (last updated 2021-12-13)

- [Information from the **WHO** about *Omicon* as a variant of concern (2021-11-26)](https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern)
- [Epidemiological report from **ECDC** Dec 10 2021](https://www.ecdc.europa.eu/en/news-events/epidemiological-update-omicron-data-10-december)
- [Epidemiological report from **ECDC** Dec 12 2021](https://www.ecdc.europa.eu/en/news-events/epidemiological-update-omicron-data-12-december)
- [Information about *Omicron* from the **CDC**](https://www.cdc.gov/coronavirus/2019-ncov/variants/omicron-variant.html)
- [Information about *Omicron* from the **Statens Serum Institute, Denmark**](https://www.ssi.dk/aktuelt/nyheder/2021/status-pa-omikron-varianten-b11529-pr-061221)
- [Statens Serum Institute, Denmark Rapport Dec 12 2021](https://covid19.ssi.dk/virusvarianter/omikron)
- [Statistics about cases of Omicron from the **Danish Statens Serum Institute**](https://files.ssi.dk/covid19/podepind-sekventering/variant-pcr-test-december2021/opgoerelse-variantpcr-covid19-13122021-j49o)
- [Information about Omicron from the **Norwegian Institute of Public Health**](https://www.fhi.no/en/id/infectious-diseases/coronavirus/statistikk-over-meldte-tilfeller-avny-virusvariant-omikron/)
- [For lineages and information related to sequences from **Cov-lineages**](https://cov-lineages.org/global_report_B.1.1.529.html)
- [Lineages and information related to sequences from **Nextstrain**](https://nextstrain.org/groups/neherlab/ncov/sweden)
- [Assembled information about the Omicron from the **PHA4GE** (Bioinformatics Pipelines & Visualization Working Group)](https://github.com/pha4ge/pipeline-resources/blob/main/docs/omicron-resources.md)
- [Europe PMC shows recent publications and preprints about Omicron see](https://europepmc.org/search?query=Omicron%20SARS-COV-2&page=2)

## Get in touch with us

Do you have suggestions or other feedback? Get in touch with us by e-mailing datacentre@scilifelab.se.
