---
title: "Omicron variant of concern: ongoing research, data, publications and preprints"
toc: true
aliases:
    - /omicron/
---

<div class="row mb-4"><div class="col col-sm-8"><ul class="nav nav-tabs nav-justified">
  <li class="nav-item">
    <a class="nav-link" href="../"><b>Variants of Concern (VoC)</b></a>
  </li>
  <li class="nav-item">
    <a class="nav-link active" href="#">Omicron VoC</a>
  </li>
</ul></div></div>

This page aims to compile information about available data, ongoing research efforts and first findings regarding the Omicron Variant of Concern (VoC) of SARS-CoV-2. Researchers and organizations from around Sweden are welcome to submit their datasets and announcements. We also present the relevant preprints and publications and links to other useful resources are provided. Please submit your suggestions using the forms below or by e-mail: datacentre@scilifelab.se. The team behind the COVID-19 Data Portal is continuously monitoring incoming suggestions and new items are published as soon as possible. Please see [this page for background on VoC information about other VoC](../).

## Available data

This section presents a list of the available data useful for research into the Omicron VoC shared by researchers affiliated to a Swedish research institute. We welcome suggestions submitted through the form below or by e-mail: datacentre@scilifelab.se. All submissions appear after the moderation by the Portal team.

{{< omicron_page_suggest_available_data >}}

<p class="text-muted">No openly available data from Swedish researchers tracked as of Dec 16.</p>

## Ongoing research efforts

We welcome announcements about ongoing research efforts from researchers and organizations in Sweden: share what you are currently working on and/or your results; look for collaborators for your project or certain tasks; look for data, equipment, reagents, etc. Any announcement useful for advacing research on Omicron is welcome. Feel free to submit the information using the form or by sending us an e-mail to datacentre@scilifelab.se.

{{< omicron_page_add_announcement >}}

<div class="alert alert-info mb-2">
  <h6>Sequences available: 10 Omicron samples</h5>
  <p class="small mb-2 text-muted">2021-12-14 15:34</p>
  <p>The Hammond lab are now running sequencing of SARS-CoV-2 variant. In our last batch of 100 samples, 10 were found to be the new Omicron. Please find the metadata record in the SciLifeLab Data Repository (doi.xxx) where we describe our work and how other researchers request access to the  bam.files. More sequences are run weekly. We are always interested in establishing new collaborations contact Dr Hammond.</p>
  <p><b>Contact:</b> Lucas Hammond, <a href="#">lucas.hammond@umu.se</a></p>
</div>

<div class="alert alert-info mb-2">
  <h6>Interested in collaborators running MS</h5>
  <p class="small mb-2 text-muted">2021-12-14 15:34</p>
  <p>A have an open call coming up and our group are interested in applying together with other labs with a similar interest in how well vaccinations protects against new variants. We are running the serology but are interested in collaborators running MS. Data Centre has kindly agreed to set interested labs in contact with me. Best regards Maria A.</p>
  <p><b>Contact:</b> Maria A, <a href="#">a.maria@gu.se</a></p>
</div>

<div class="alert alert-info mb-2">
  <h6>Looking for in-house/commercial primers for detection of Omicron with RT-qPCR</h5>
  <p class="small mb-2 text-muted">2021-12-14 12:20</p>
  <p>The Andersson Lab has been working on influenza since 2010. Recent development of the pandemic and the emergence or another SARS-CoV2- variant has made us decide to start working on variant detection. Would you be willing to share “in-house” made/commercial primers for detection of Omicron with RT-qPCR?. We hope to get our work up and running before Christmas. Data Centre can help set us in contact. Thank you!</p>
  <p><b>Contact:</b> Niclas Andersson, <a href="#">niclas.andersson@uu.se</a></p>
</div>

<!-- <div class="card mb-2">
  <div class="card-body">
    <h5 class="card-title">This is the title of the announcement</h5>
    <h6 class="card-subtitle mb-2 text-muted">2021-12-14 15:34</h6>
    <p class="card-text">This is where the text of the announcement goes. The Swedish COVID-19 Data Portal provides information, guidelines, tools and services to support researchers to utilise Swedish and European infrastructures for data sharing. The portal is a national node of the European COVID-19 Data Portal.</p>
    <a href="#" class="card-link">name.surname@uu.se</a>
  </div>
</div> -->

## Relevant publications and preprints

Here we display a selection of preprints and journal publications reporting results on Omicron VoC.

{{< omicron_page_pubs >}}

## Exploring Omicron VoC in Sweden using Nextstrain

<figure class="figure float-right mx-3 w-50">
  <img src="/voc/nextstrain_build.png" alt="Screenshot from the Nextstrain build" class="img-thumbnail">
  <figcaption class="figure-caption mt-1"><i>Screenshot from the Nextstrain build</i></figcaption>
</figure>

There is a [Nextstrain build maintained by the Neher lab and NextStrain team](https://nextstrain.org/groups/neherlab/ncov/sweden) that focusses specifically on sequences from Sweden, follow [this link for Omicron focus](https://nextstrain.org/groups/neherlab/ncov/sweden?f_clade_membership=21K%20%28Omicron%29). Please keep in mind that quality of the analyses by Nextstrain depends on the amount and quality of available data so the results should be taken with caution.

There are 4 main visualisation panels in Nextstrain:

- A phylogenetic tree showing the relationships between variants.
- A geographic map showing which variants have been detected in Sweden and their relative prevalence by default. Other aspects of the data related to sequences (e.g. the sampling date) can be displayed instead using the **Color By** dropdown menu to the left of the visualisations panel. **NOTE:** Some data are visible for other countries, as a sample from all over the world is needed to properly root the phylogenetic tree (see [here](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/data-prep.html) for information on subsampling procedures).
- A 'diversity' plot that shows information related to differences in amino acids and nucleotides across variants.
- A frequency plot showing the relative dominance of variants.

For more information on what is shown in the panels, please see e.g. this [2018 paper by Hadfield et al.](https://academic.oup.com/bioinformatics/article/34/23/4121/5001388), or [this section on interpreting analyses](https://docs.nextstrain.org/en/latest/learn/interpret/index.html) in the Nextrain documentation. Further, to understand how different visuaalisations relate to each other, see [the narrative written by Hodcroft et al. in 2020](https://nextstrain.org/narratives/trees-background).

## Other useful resources

Here we list a number of useful resources related to the new *SARS-CoV-2 Omicron variant* (last updated 2021-12-13)

- [Information from the **WHO** about *Omicon* as a variant of concern (2021-11-26)](https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern)
- [Epidemiological report from **ECDC** Dec 10 2021](https://www.ecdc.europa.eu/en/news-events/epidemiological-update-omicron-data-10-december)
- [Epidemiological report from **ECDC** Dec 12 2021](https://www.ecdc.europa.eu/en/news-events/epidemiological-update-omicron-data-12-december)
- [Information about *Omicron* from the **CDC**](https://www.cdc.gov/coronavirus/2019-ncov/variants/omicron-variant.html)
- [Information about *Omicron* from the **Statens Serum Institute, Denmark**](https://www.ssi.dk/aktuelt/nyheder/2021/status-pa-omikron-varianten-b11529-pr-061221)
- [Statens Serum Institute, Denmark Rapport Dec 12 2021](https://covid19.ssi.dk/virusvarianter/omikron)
- [Statistics about cases of Omicron from the **Danish Statens Serum Institute**](https://files.ssi.dk/covid19/podepind-sekventering/variant-pcr-test-december2021/opgoerelse-variantpcr-covid19-13122021-j49o)
- [Information about Omicron from the **Norwegian Institute of Public Health**](https://www.fhi.no/en/id/infectious-diseases/coronavirus/statistikk-over-meldte-tilfeller-avny-virusvariant-omikron/)
- [For lineages and information related to sequences from **Cov-lineages**](https://cov-lineages.org/global_report_B.1.1.529.html)
- [Lineages and information related to sequences from **Nextstrain**](https://nextstrain.org/groups/neherlab/ncov/sweden)
- [Assembled information about the Omicron from the **PHA4GE** (Bioinformatics Pipelines & Visualization Working Group)](https://github.com/pha4ge/pipeline-resources/blob/main/docs/omicron-resources.md)
- [VEO report on mutations and variation in publicly shared SARS-CoV-2 raw sequencing data, Report No. 10, 15 December 2021](https://www.veo-europe.eu/news/nyhed?id=%7B1B87370A-0EBB-4412-81E5-9390A6FCDC07%7D)

## Get in touch with us

Do you have suggestions or other feedback? Get in touch with us by e-mailing datacentre@scilifelab.se.
