---
title: "Genomic Pandemic Preparedness Portfolio (G3P) – Nationally scalable genomics portfolio for detection and surveillance of viral outbreaks"
category: "plp1"
banner: "/resources_thumbs/g3p.jpg"
resource_info:
  name: "Genomic Pandemic Preparedness Portfolio (G3P) – Nationally scalable genomics portfolio for detection and surveillance of viral outbreaks"
  pi: Valtteri Wirta
  host_organisation: "Karolinska Institutet"
  contact: "Valtteri Wirta<br>Director, Clinical Genomics facility<br>Email: [valtteri.wirta@ki.se](mailto:valtteri.wirta@ki.se)"
for_background_table:
  pi: Valtteri Wirta
  pi_affiliation: Karolinska Institutet
---

The current SARS-CoV-2 pandemic highlights the need of building pandemic preparedness capabilities for genetic monitoring (surveillance) of pathogens causing pandemics, as well as early detection of emerging pathogens and antimicrobial resistance in future pandemics and outbreaks. Therefore, our project focus on sequencing-based pandemic preparedness, by developing laboratory assays, along with associated bioinformatics analysis workflows, data visualisation for knowledge sharing and interpretation, and other tools for efficient national coverage.

Developed assays and tools will be established across the SciLifeLab´s Clinical Genomics nodes, Genomic Medicine Centers, and clinical microbiology laboratories to reach national coverage, minimise the distance to point-of-care, and ensuring rapid turnaround times. Methods can aid in testing critically ill patients with diffuse symptoms in non-pandemic situations, and assays will contribute to the SciLifeLab Precision Medicine capability.

Specific aims are to:

1. Maximise the effects of ongoing efforts by <b>strengthening the national networks</b> and collaborations relevant for pandemic preparedness.

2. Continue developing <b>whole-genome sequencing-based pathogen surveillance</b> by driving national harmonisation and developing tools for data aggregation and visualisation.

3. Establish <b>validated metagenomic assays</b> for detection of known and unknown viruses and methods for detecting antimicrobial resistance development.

4. <b>Coordinate data handling</b> i.e. collection, annotation, structuring and aggregation.

5. <b>Provide the developed infrastructure as a services</b> regarding wet-lab data generation, bioinformatic analysis, biobanking, and data standardisation, for the research community, healthcare and public health authorities.

This resource involves three subprojects:

###### 1. Highly parallelised multi-taxonomic profiling of shotgun metagenomic data (taxprofiler)

For more information on Taxprofiler (including information on access, use, and contact), please see the dedicated [Taxprofiler](/resources/taxprofiler/).

###### 2. SARS-CoV-2 bioinformatics pipeline – GMS Artic

For more information on GMS Artic (including information on access, use, and contact), please see the dedicated [GMS Artic page](/resources/gms-artic/).

###### 3. SARS-CoV-2 Reporter for visualisation of SARS-CoV-2 genome sequence data (SC2reporter)

For more information on SC2reporter (including information on access, use, and contact), please see the dedicated [SC2reporter page](/resources/sc2reporter/).

##### Research findings:

Imagine a world where scien#sts can track and respond to pandemics in real-#me. That’s exactly what the Genomics-based Pandemic Preparedness PorKolio (G3P) ini#a#ve is working towards. One of its biggest achievements is building a na#onal network of experts and organiza#ons dedicated to using genomic science to stay ahead of infec#ous diseases. This network includes key research projects and public health agencies, such as the RAPID-SEQ project (focused on fast sequencing of viruses), the Swedish Public Health Agency, and the EU-funded Infragen network, which is pioneering nanopore sequencing to quickly analyze the gene#c makeup of SARS-CoV-2 across Europe.

Another major milestone is the Na#onal Pandemic Center’s biobank, which now holds over 1.4 million biological samples—an invaluable resource for tracking virus evolu#on. To support this, all the necessary legal agreements have been put in place, ensuring smooth collabora#on.

Data is the backbone of genomic research, and G3P has made sure that informa#on about SARS-CoV- 2 is analyzed in a standardized way. These results are shared instantly with both Sweden’s na#onal health database (GENSAM) and the global virus tracking system (GISAID)—helping scien#sts worldwide.

Beyond COVID-19, G3P is also leading efforts in metagenomics, a field that allows researchers to study all the microbes in a sample at once. To improve methods, we conducted a na#onwide survey of the tools and techniques currently in use, ensuring that Sweden stays at the forefront of microbial surveillance. By tes#ng different sequencing approaches, from short-read to long-read technologies, we are working towards a more powerful and unified system for detec#ng and studying pathogens.

In short, G3P is not just about responding to pandemics—it’s about staying ahead of them. By building strong collabora#ons, refining technology, and ensuring real-#me data sharing, this ini#a#ve is helping to create a future where outbreaks can be tackled faster and more effec#vely than ever before.

##### Impact on prepardness for future pandemics:

<b>National Laboratory Network for Pandemic Preparedness</b>:
We have successfully integrated all seven university hospitals and key stakeholders—including the Public Health Agency of Sweden (FoHM), Genomic Medicine Sweden (GMS), the National Pandemic Center (NPC), and the SciLifeLab Clinical Genomics platform—into a resilient national laboratory network. To enable rapid scale-up and efficient sample acquisition, this work is conducted in close collaboration with the SciLifeLab National Genomics Infrastructure (NGI).

<b>Advancing Surveillance Capabilities</b>:
The G3P project has significantly strengthened Sweden’s capacity for pandemic preparedness by establishing high-throughput, sequencing-based surveillance systems. This initiative has also fostered a na#onwide collabora#on among laboratories utilizing nanopore technology, enhancing Sweden’s ability to rapidly develop and deploy new sequencing methods in response to emerging outbreaks.

Through regular meetings and structured information exchange, this network facilitates knowledge- sharing, technical troubleshooting, and continuous methodological improvements, ultimately enhancing Sweden’s pandemic response capabilities. These efforts translate into faster diagnostics, improved coordination, and more effecive public health interventions during infectious disease outbreaks.

In addition, several new protocols critical for pandemic preparedness have been collaboratively developed. The close partnership between Clinical Genomics nodes and NGI enables the rapid implementation and scaling of diagnostic assays, ensuring a surge capacity that extends beyond routine healthcare opera#ons. This capability is crucial for meeting increased diagnostic demands during future pandemics.

To further strengthen national laboratory preparedness, we have expanded biobanking activities through the NPC, involving biobank coordinators from all Swedish regions as well as Biobank Sweden. A dedicated SciLifeLab-hosted website has been developed to provide interactive standard operating procedures (SOPs) and decision flowcharts, facilitating optimized and rapid sequencing workflows during pandemic situations. Additionally, key resources and tools are disseminated via The Swedish Pathogens Portal, ensuring broad accessibility.

<b>Metagenomics</b>:
The findings of this project have substan#al implications for infectious disease diagnostics and pandemic preparedness. By advancing microbial detection efficiency, this initiative contributes to faster and more cost-effective pathogen identification, leading to earlier clinical interventions and potential to reduced healthcare expenditures. Improved diagnostic accuracy enables more targeted treatment strategies, reducing reliance on broad-spectrum antibiotics—a key factor in mitigating the global threat of antimicrobial resistance (AMR).

Furthermore, the bioinformatics workflow Taxprofiler, developed partly within this project, is part of an international collaborative effort, ensuring data interoperability and robustness. This cross-border cooperation strengthens pipeline development and validation, providing a scalable framework for future pathogen surveillance. The SARS-CoV-2 pandemic underscored the critical importance of international collaboration in bioinforma#cs, demonstrating its role in accelerating innovation and enhancing global pandemic response capabilities.