---
title: "Gothenburg initiative for pandemic laboratory preparedness (GILP)"
category: "tdp"
resource_info:
    name: "Gothenburg initiative for pandemic laboratory preparedness (GILP)"
    pi: Magnus Lindh
    host_organisation: Dept. of Clinical Virology, University of Gothenburg and Clinical Microbiology, Sahlgrenska University Hospital
    contact: "Magnus Lindh<br>Dept. of Clinical Virology, University of Gothenburg and Clinical Microbiology, Sahlgrenska University Hospital<br>Email: [magnus.lindh@microbio.gu.se](mailto:magnus.lindh@microbio.gu.se)"
for_background_table:
    pi: Magnus Lindh
    pi_affiliation: Gothenburg University
    lab: Sahlgrenska University Hospital
---

Clinical Microbiology, Sahlgrenska University Hospital (CMSU) is one of the largest microbiology laboratories in northern Europe. It provides microbiological diagnostics to a population of 1.2 million people and collaborates with the Institute of Biomedicine, University of Gothenburg, on a broad range of research projects, including several on Covid-19. During the Covid-19 pandemic CMSU was the first hospital laboratory in Sweden to offer SARS-CoV-2 RNA testing and the first to provide test results on mobile phones.

We have resources such as deep sequencing instruments and bioinformatics, but further development is important for pandemic laboratory preparedness. For this aim, established routines that provide us with waste water specimen and clinical samples, are important. The technologies and expertise in ongoing research are also critical.

We will use and develop these resources further to establish a pandemic preparedness. Specifically we will:

1. Develop and expand our metagenomics microbial monitoring on waste water to include broad searches for emerging pathogens. This will be achieved by improved enrichment of viral nucleic acids, greater sequencing depth and refined bioinformatics searches.

2. Develop and expand metagenomics microbial monitoring on clinical samples by similar strategies.

3. Refine a bioinformatics tool for genomic signatures.

4. Develop bioinformatic search algorithms for minor viral subtypes,

5. Expand a high-throughput T cell assay pipeline for additional viruses.

6. Further develop our BSL3 competence for viral culture, neutralisation and antiviral assays.
