---
title: "Infections in travelers"
category: "test"
banner: "/resources_thumbs/travel.jpg"
resource_info:
  name: "Infections in travelers"
  pi: Tobias Allander
  host_organisation: Clinical Microbiology at Karolinska University Hospital
  contact: "Tobias Allander<br>Dept. of Clinical Microbiology, Karolinska University Hospital and Dept. of Microbiology, Tumor, and Cell Biology, Karolinska Institutet<br>Email: [tobias.allander@regionstockholm.se](mailto:tobias.allander@regionstockholm.se)"
for_background_table:
  pi: Tobias Allander
  pi_affiliation: Karolinska Institutet
  lab: Karolinska University Hospital
---

Correct diagnosis of infectious diseases is the foundation of infection control and surveillance. There are excellent diagnostic tools for common infections, but rare or unexpected infectious agents are more difficult to identify, and risk being missed. A more widespread use of unbiased diagnostic methods, such as metagenomic sequencing, would therefore improve surveillance and early identification of pandemic threats, including known and unknown pathogens.

With the support of PLP grants, we have developed a diagnostic metagenomic sequencing pipeline, which is now available for clinical diagnostic use. The main intended areas of use are severe, unexplained infections (CNS, respiratory or generalised), particularly cases when unusual pathogens are more likely, such as in immunosuppressed patients and travelers from tropical regions. A study evaluating the diagnostic yield in immunosuppressed patients with pneumonia is ongoing. The present study proposal aims for evaluation of metagenomic sequencing in patients suffering from febrile infections following travel in tropical countries.

The “Tropical fever study” at Karolinska University Hospital has included 521 patients seeking care for fever after travel in the tropics during 2017-2020. Approximately half of the cases lack etiological diagnosis. We will analyse an estimated 100 selected stored serum samples by metagenomic sequencing, with the aims to evaluate the diagnostic value of metagenomic sequencing of serum samples in this patient group, to identify additional causative microorganisms, and to describe the microbial genomes present in the serum of patients with fever of unknown etiology. We will also investigate paired acute and convalescent serum samples by a subtractive bioinformatic approach, to identify sequences associated with the disease episode independently from taxonomic classification.

Improved diagnosis of rare infections in travelers is important for early identification of pandemic threats.

##### Research findings:

- Diagnostic metagenomic sequencing (unbiased sequencing of all nucleic acid in thesample) is now in routine clinical use
- The diagnostic performance of metagenomic sequencing for different sample types and different viruses has been thoroughly evaluated
- As a proof of concept, metagenomic sequencing has been applied to previously collected samples from clinical studies, including pediatric cancer patients and travelers returning from tropical regions. Results show that several previously
undetected infections were identified, and a previously unknown viral genome was characterized.
- We have shown that relevant information about the immune response of the patient can be collected from metagenomic sequencing results.

##### Impact on prepardness for future pandemics:

Early and rapid detection of pandemic threats, e.g. zoonotic infections or emerging virus variants is a key element of pandemic preparedness. It is therefore important that general and unbiased diagnostic methods become more widely available, more rapid and less costly to perform. This goal has been recognized by the European Health Emergency Preparedness and Response Authority (HERA). Implementation and continued development of diagnostic metagenomic sequencing therefore directly contributes to pandemic preparedness.

Diagnostic metagenomic sequencing results could also provide insights into the immune response of the patients, which is important for treating severe infections. This can be exemplified by the introduction of corticosteroid treatment for severe covid-19 during the pandemic. Such advances of treatment can potentially be accelerated by the help of
metagenomic sequencing.

Out proof-of-concept studies have illustrated how metagenomic sequencing often identifies otherwise overlooked infectious agents. We have also described a previously unknown virus genome.

Finally, the dealing with high throughput sequencing and the vast variety of microbiological findings coming out of it contributes to building a broad technical and microbiological competence in the diagnostic laboratories, which will strengthen pandemic preparedness.
