---
title: "Metagenomic sequencing for pathogen identification and analysis"
category: "tdp"
banner: "/resources_thumbs/metagenomic-seq.jpg"
resource_info:
  name: "Metagenomic sequencing for pathogen identification and analysis"
  pi: Jenny Welander
  host_organisation: Clinical Microbiology at Linköping University Hospital
  contact: "Jenny Welander<br>Clinical Microbiology at Linköping University Hospital<br>Email: [jenny.welander@regionostergotland.se](mailto:jenny.welander@regionostergotland.se)"
for_background_table:
  pi: Jenny Welander
  pi_affiliation: Linköping University
  lab: Linköping University Hospital
---

Clinical Microbiology at Linköping University Hospital plans to develop a capability to quickly identify and characterise unknown pathogens through metagenomic sequencing. This will enable detection and sequence analysis of pathogens directly from clinical samples of unknown etiology, without the need of pathogen-specific reagents.

Since the middle of June 2022, an M.Sc. in engineering biology has been recruited for the project and is currently learning the already developed methods for whole-genome sequencing of bacteria and target enrichment sequencing of SARS-CoV-2, with supervision from a biologist and a bioinformatician involved in the project.

In parallel, we have started testing three different RNA extraction techniques to optimise RNA isolation from nasopharynx samples.

A literature review has been performed regarding library preparation, and as a result, we have ordered library preparation kits for metagenomic RNA-seq on Illumina and Nanopore. These will be tested and evaluated on human samples with varying concentrations of known viruses (influenza, RS and SARS-CoV-2).

Different bioinformatic approaches will be developed, evaluated and optimised. Once the laboratory and bioinformatic methods are validated on samples of known content, they will be further tested on samples from patients with unknown infections. With this technology development, the laboratory will have an increased preparedness to quickly analyse novel pathogens and to follow and quickly respond to future pandemic situations.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. With this funding, the capability aimed to further develop the methodology into a clinical routine method for analysis of samples of unknown etiology. This will include the addition of CNS pathogens in cerebrospinal fluid samples. The capability also aimed to establish quicker sequencing using long-read technology to decrease the turn-around-time in cases where time is essential for patient outcome. Further development was also planned for bioinformatic pipelines and databases, in part to allow more samples to be analysed by medical doctors, bioinformaticians, and biologists to build experience with interpreting complex results.
