---
title: "Metagenomic sequencing for pathogen identification and analysis"
category: "tdp"
banner: "/resources_thumbs/metagenomic-seq.jpg"
resource_info:
  name: "Metagenomic sequencing for pathogen identification and analysis"
  pi: Jenny Welander
  host_organisation: Clinical Microbiology and Precision Medicine Laboratory, Linköping University Hospital
  contact: "Jenny Welander<br>Clinical Microbiology and Precision Medicine Laboratory, Linköping University Hospital<br>Email: [jenny.welander@regionostergotland.se](mailto:jenny.welander@regionostergotland.se)"
for_background_table:
  pi: Jenny Welander
  pi_affiliation: Linköping University
  lab: Linköping University Hospital
---

Metagenomic sequencing enables the detection of unusual or unexpected pathogens in patient samples where routine methods fail. In a collaboration between Clinical Microbiology and the Precision Medicine Laboratory at Linköping University Hospital, methodology for metagenomic sequencing has been developed and evaluated.

The newly developed capability enables sequencing of total RNA and DNA from patient samples to detect bacteria, viruses, fungi, and parasites without any need for culture or amplification with pathogen-specific reagents. The method is based on nucleic acid extraction, parallel RNA and DNA library preparation, and Illumina sequencing, followed by bioinformatic taxonomic classification using the <a href="/resources/taxprofiler/">nf-core/taxprofiler</a> pipeline.

The method will be used to detect and analyse unusual pathogens in respiratory samples and skin lesions. In addition, the method will be used for the analysis of cerebrospinal fluid in CNS infections that cannot be diagnosed using other methods. With this technology development, the laboratory has an increased preparedness to monitor and quickly respond to future outbreaks and pandemics.

The capability is participating in a national collaboration within the Genomic Medicine Sweden (GMS) network to coordinate laboratory and bioinformatics development, and exchange control samples.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see the <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. With this funding, the methodology is now being further developed and validated into a clinical routine method for analysis of samples of unknown etiology. So far, validation of the method has included RNA viruses, DNA viruses, bacteria, and fungi. Biologists, bioinformaticians, and medically responsible doctors are working together to further optimise the method and build experience around the interpretation of complex results. Additional bioinformatics tools are being evaluated to improve pathogen characterisation and to enable the analysis of antimicrobial resistance. The capability also aims to establish quicker sequencing using long-read technology to decrease the turn around time.


##### Research findings:

In this technology development project, a complete, clinically useful method has been developed for analyzing the total genetic material in patient samples. The metagenomic approach enables the identification of all types of pathogens, including viruses, bacteria, fungi and parasites, in cases where current routine methods fail. As a result, the Clinical Microbiology laboratory has gained an increased capability of diagnosing unusual infections in severely ill patients, as well as an increased preparedness to monitor and quickly respond to future outbreaks and pandemics.

##### Impact on prepardness for future pandemics:

The metagenomics methodology developed in this project will enable identification of rare and unexpected pathogens as well as pathogens with novel mutations or newly acquired genes, in cases where current routine methods in the Clinical Microbiology laboratory are insufficient. The developed methods can also provide information about genotypes (for outbreak investigation and surveillance), potential drug resistance genes and virulence factors, as well as information about co-infections with multiple pathogens. Our continuous development and maintenance of the laboratory methods and bioinformatics will ensure that expertise, sequencing capacity, materials and protocols are kept up-to-date and ready for new outbreaks and pandemic situations. Through participation in metagenomics working groups and PLP tests and networks, we have established national collaborations that ensure sharing of protocols and data, and further increase our preparedness to future pandemics.