---
title: "Identification of molds and yeasts using Nanopore sequencing and MALDI-TOF."
category: "test"
banner: "/resources_thumbs/mould_yeast.jpg"
resource_info:
  name: "Identification of molds and yeasts using Nanopore sequencing and MALDI-TOF."
  pi: Johan Lindh
  host_organisation: Uppsala University
  contact: "Johan Lindh<br>Department of Medical Sciences, Clinical Microbiology, Uppsala University<br>Email: [johan.lindh@akademiska.se](mailto:johan.lindh@akademiska.se)"
for_background_table:
  pi: Johan Lindh
  pi_affiliation: Uppsala University
---

The department of Clinical microbiology and hygiene (KMB) at Uppsala university hospital adopted to the pandemic situation very fast, we were up and running with in-house established methods for RT-PCR analyses of SARS-CoV2 already in February, 2020. We have since then developed several different methods including sequencing protocols using Nanopore technology and mutantRTPCR.

We have today a large capacity in the area of SARS-CoV-2 analyses, both whole genome sequencing and in RTPCR and are also ready to detect new viruses. We have developed qualified methods and the use of 96 hole plates in all steps of above. Since all methods and instruments are validated and quality assured they are also used in other routine diagnosis. Our future perspectives in this area are to develop 18S sequencing methods in order to identifye molds and yeasts together with protocols to detect the same using MALDI-TOF.

Another method which can detect different microbes at a high output is the MALDI-TOF system. Today we have the system running at a capacity of 96 samples per hour and per instrument. A big advantage with this system, if compared with RTPCR, is that the detection is at the level of expressed proteins instead of genetic material. This is especially true if resistance to drugs will be measured.

##### Research findings:

*Whole genome sequencing based on the Nanopore sequencing*: Protocols have been developed, including extraction and sequencing. Test runs using Candida albicans will be performed during spring 2025. Flow chart of bioinformatic procedures are established.

*18S sequence*: Protocols have been developed, including extraction, sequencing and bioinformatic flow chart. The method will be running as a routine method during autumn 2025.

##### Impact on prepardness for future pandemics:

The incidence of humans getting infected by yeast and, or mold is increasing, both globally and locally. Today, many of the different analyzes are performed by growing the microbes and by eye, microscopes and expertise evaluate the results and give the customer an accurate answer. The methods have several advantages, such as a low price. However, while bacteria very often have a relatively short generation time, results can often be visualized within 24 hours after they have reached the laboratory. Furthermore, due to history and the increasing threat of different resistant bacteria that take over health care, the response has been the development of better and faster methods. Most of the new methods are developed and during development use the short generation time of bacteria as an advantage. These advantages cannot be used if mold or yeast are to be diagnosed since they have a much longer generation time. Diagnosing a mold or yeast can take up to a week and if a resistance marker is needed, sometimes even longer. This time might be too long for the patients before correct treatment can be distributed.

The major aim of this project is to decrease the time between a sample is taken until an answer is given to the clinic and that the answer should be more accurate and contain more information about possible resistance.

We believe that the next pandemic could be caused by fungi, with complex resistance to available treatment and, therefore, it is of great importance that clinical laboratories are prepared with accurate and fast methods to detect the causative fungi.
