---
title: "Next generation clinical virology"
category: "tdp"
banner: "/resources_thumbs/next-gen-clin-microbio.jpg"
resource_info:
  name: "Next generation clinical virology"
  pi: Tobias Allander
  host_organisation: Clinical Microbiology at Karolinska University Hospital
  contact: "Tobias Allander<br>Dept. of Clinical Microbiology, Karolinska University Hospital and Dept. of Microbiology, Tumor, and Cell Biology, Karolinska Institutet<br>Email: [tobias.allander@regionstockholm.se](mailto:tobias.allander@regionstockholm.se)"
for_background_table:
  pi: Tobias Allander
  pi_affiliation: Karolinska Institutet
  lab: Karolinska University Hospital
---

Next generation sequencing offers numerous possibilities for improved diagnostics and management of infectious diseases. A comprehensive goal for Clinical Microbiology at Karolinska is to apply viral metagenomics and human immune transcriptomics in healthcare, and develop the following capabilities:

- Diagnostic metagenomic sequencing.
- Metagenomic virus surveillance of respiratory samples.
- Characterisation of unclassifiable metagenomic sequences.
- Host immune transcriptome based microbial diagnostics.
- Immunotherapy guidance.

Activities so far have focused on establishing diagnostic metagenomic sequencing with a focus on viral infections. Wet lab protocols have been developed for Illumina and Nanopore sequencing and a first bioinformatics pipeline is in place. Further development is ongoing within the Genomic Medicine Sweden (GMS) collaboration. We have analysed 25 samples (respiratory tract samples, serum, cerebrospinal fluid) with known viral findings and compared metagenomics by both Illumina (25M reads) and Nanopore sequencing with real-time PCR results. Results indicate that the sensitivity of metagenomic sequencing by Illumina is better than our target value corresponding to Ct 30. Nanopore sequencing as expected has somewhat lower sensitivity but has proven useful because of its short turn-around time. We are appointed a reference center by ECDC for metagenomic sequencing of non-A-E hepatitis cases.

We have also made progress in the characterisation of unclassified genomes, where we are currently focusing on two previously unknown picornavirus species. Future activities will be to complete the validation of diagnostic metagenomic sequencing, release the assay to health care and set up a clinical study protocol for the received samples, so that the data can be used for research. We will also initiate immune transcriptome studies.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. With this funding, the capability will further develop the wet lab and bioinformatics protocols for metagenomic sequencing, Nanopore-based metagenomic sequencing, and transcriptome profiling as a diagnostic tool. It will also evaluate the diagnostic value of mNGS and the collection of data through clinical study protocols, initiate virus surveillance, and develop new virus classification tools.

##### Research findings:

- Diagnostic metagenomic sequencing (unbiased sequencing of all nucleic acid in thesample) is now in routine clinical use
- The diagnostic performance of metagenomic sequencing for different sample types and different viruses has been thoroughly evaluated
- As a proof of concept, metagenomic sequencing has been applied to previously collected samples from clinical studies, including pediatric cancer patients and travelers returning from tropical regions. Results show that several previously
undetected infections were identified, and a previously unknown viral genome was characterized.
- We have shown that relevant information about the immune response of the patient can be collected from metagenomic sequencing results.

##### Impact on prepardness for future pandemics:

Early and rapid detection of pandemic threats, e.g. zoonotic infections or emerging virus variants is a key element of pandemic preparedness. It is therefore important that general and unbiased diagnostic methods become more widely available, more rapid and less costly to perform. This goal has been recognized by the European Health Emergency Preparedness and Response Authority (HERA). Implementation and continued development of diagnostic metagenomic sequencing therefore directly contributes to pandemic preparedness.

Diagnostic metagenomic sequencing results could also provide insights into the immune response of the patients, which is important for treating severe infections. This can be exemplified by the introduction of corticosteroid treatment for severe covid-19 during the pandemic. Such advances of treatment can potentially be accelerated by the help of
metagenomic sequencing.

Out proof-of-concept studies have illustrated how metagenomic sequencing often identifies otherwise overlooked infectious agents. We have also described a previously unknown virus genome.

Finally, the dealing with high throughput sequencing and the vast variety of microbiological findings coming out of it contributes to building a broad technical and microbiological competence in the diagnostic laboratories, which will strengthen pandemic preparedness.