---
title: "Next generation clinical virology"
category: "tdp"
banner: "/resources_thumbs/next-gen-clin-microbio.jpg"
resource_info:
  name: "Next generation clinical virology"
  pi: Tobias Allander
  host_organisation: Clinical Microbiology at Karolinska University Hospital
  contact: "Tobias Allander<br>Dept. of Clinical Microbiology, Karolinska University Hospital and Dept. of Microbiology, Tumor, and Cell Biology, Karolinska Institutet<br>Email: [tobias.allander@regionstockholm.se](mailto:tobias.allander@regionstockholm.se)"
for_background_table:
  pi: Tobias Allander
  pi_affiliation: Karolinska University
  lab: Karolinska University Hospital
---

<div class="alert alert-info">
    <i class="bi bi-info-circle-fill"></i> This project received continuation funding from the REPLPCM call, see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information.
</div>

Next generation sequencing offers numerous possibilities for improved diagnostics and management of infectious diseases. A comprehensive goal for Clinical Microbiology at Karolinska is to apply viral metagenomics and human immune transcriptomics in healthcare, and develop the following capabilities:

- Diagnostic metagenomic sequencing.
- Metagenomic virus surveillance of respiratory samples.
- Characterization of unclassifiable metagenomic sequences.
- Host immune transcriptome based microbial diagnostics.
- Immunotherapy guidance.

Activities so far have focused on establishing diagnostic metagenomic sequencing with a focus on viral infections. Wet lab protocols have been developed for Illumina and Nanopore sequencing and a first bioinformatics pipeline is in place. Further development is ongoing within the Genomic Medicine Sweden (GMS) collaboration. We have analyzed 25 samples (respiratory tract samples, serum, cerebrospinal fluid) with known viral findings and compared metagenomics by both Illumina (25M reads) and Nanopore sequencing with real-time PCR results. Results indicate that the sensitivity of metagenomic sequencing by Illumina is better than our target value corresponding to Ct 30. Nanopore sequencing as expected has somewhat lower sensitivity but has proven useful because of its short turn-around time. We are appointed a reference center by ECDC for metagenomic sequencing of non-A-E hepatitis cases.

We have also made progress in the characterization of unclassified genomes, where we are currently focusing on two previously unknown picornavirus species. Future activities will be to complete the validation of diagnostic metagenomic sequencing, release the assay to health care and set up a clinical study protocol for the received samples, so that the data can be used for research. We will also initiate immune transcriptome studies.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. With this funding, the capability will further develop the wet lab and bioinformatics protocols for metagenomic sequencing, Nanopore-based metagenomic sequencing, and transcriptome profiling as a diagnostic tool. It will also evaluate the diagnostic value of mNGS and the collection of data through clinical study protocols, initiate virus surveillance, and develop new virus classification tools.
