---
title: "Development of a targeted metagenomic NGS platform for diagnostics and epidemiological surveillance of viral pandemic threats"
category: "tdp"
banner: "/resources_thumbs/ngs-diagnositics.jpg"
resource_info:
  name: "Development of a targeted metagenomic NGS platform for diagnostics and epidemiological surveillance of viral pandemic threats"
  pi: Lisa Pettersson
  host_organisation: Dept. of Clinical Microbiology, Umeå University Hospital & Dept. of Clinical Microbiology, Umeå University
  contact: "Lisa Pettersson<br>Dept. of Clinical Microbiology, Umeå University Hospital & Dept. of Clinical Microbiology, Umeå University<br>Email: [Lisa.pettersson@regionvasterbotten.se](mailto:Lisa.pettersson@regionvasterbotten.se)"
for_background_table:
  pi: Lisa Pettersson
  pi_affiliation: Umeå University
  lab: Umeå University Hospital
---

The aim of this project is to develop and implement a targeted metagenomic next-generation sequencing-based (mNGS) platform that utilises a hybrid-capture and amplification approach for sensitive and accurate detection of viral pathogens. The focus will be on an enrichment panel that can detect > 3,000 viral species.

<b>Project outline</b>

The proposed system includes implementation of validated methods for:

1. Constructing Illumina TruSeq sequencing libraries
2. Performing target-capture using Twist BioScience oligomer panels
3. Performing post-capture library amplification
4. Sequencing on Illumina instruments (MiniSeq, MiSeq, NextSeq)
5. Automated, reproducible bioinformatics workflow (NextFlow)
6. Data storage and management
7. User-friendly presentation of results (secure web application, LDAP/AD user authentication, database backend)
8. Submission of data generated as part of consented academic studies to the ScliLifeLabs Data Centre in adherence with open science standards (FAIR)

<b>Project development</b>

We have hired a laboratory technician that is under training to perform the laboratory work needed.
We are developing a web application that can translate and present the NGS data as clinically relevant information. The application must be sufficiently user-friendly from a clinical point of view to allow the platform to be used in clinical routine. The platform will, however, also be used for projects from the academic research community.

We plan to analyse clinical samples with common viruses (for example Influenza A and calicivirus) and samples spiked with inactivated viral species constituting possible pandemic threats to investigate the performance of the panel.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. With this funding, the capability will focus on the implementation of the metagenomic NGS platform and develop a workflow for clinical use, both for diagnostics and surveillance. Effort will also be put into reaching out to clinicians to inform of the new possibilities that the NGS platform has to offer as a novel diagnostic tool for viral diseases – including unknown viruses.

##### Research findings:

We have found that the virus targeting sequencing method (including wet lab and bioinformatic pipeline) can detect very low numbers of virus in patient samples. The method can detect viruses present in patient samples from various locations, such as serum, plasma, cerebrospinal fluid (CSF), and vesicle samples. The methodology can not only detect viruses at the species level, but it can also, in many cases, generate whole genome sequences. Thus, it is suitable for outbreak investigations. The bioinformatic pipeline we developed is working as anticipated; sequence data is transformed into interactive graphs and data tables that are easily interpretable by clinicians.

In a collaboration we have also searched for viruses in ticks from the African wildlife, where our methodology could detect non-human viruses. This demonstrates that the method is capable of detecting animal viruses with zoonotic potential, which may cross the species barrier and emerge as human pathogens

##### Impact on prepardness for future pandemics:

Our targeted metagenomic NGS platform will hopefully be used to screen a large number of samples in the search for new or emerging viruses. The screening of samples can also be used for molecular surveillance to identify new viral strains of already circulating viruses for example, SARS-CoV-2 and Influenza A. This enables the identification of specific strains
with mutations that may impact the virus transmissibility or resistance to antivirals. Screening for emerging viruses and new viral strains can provide early warnings of potential epidemics or pandemics by identifying viruses before they lead to large-scale outbreaks.