---
title: "Rapid establishment of comprehensive laboratory pandemic preparedness – RAPID-SEQ"
category: "plp1"
banner: "/resources_thumbs/rapid-seq.jpg"
resource_info:
  name: "Rapid establishment of comprehensive laboratory pandemic preparedness – RAPID-SEQ"
  funded_project_title: "Rapid establishment of comprehensive laboratory pandemic preparedness – RAPID-SEQ"
  pi: Jan Albert
  host_organisation: "Karolinska Institutet"
  contact: "Jan Albert<br>Professor<br>Email: [jan.albert@ki.se](mailto:jan.albert@ki.se)"
for_background_table:
  pi: Jan Albert
  pi_affiliation: Karolinska Institutet
---

The constant emergence and replacement of new _SARS-CoV-2 variants_ has shown the need for continuous large-scale surveillance to monitor for future pandemic threats. This project builds on established collaboration between _SciLifeLab_ and the _Karolinska University Laboratory_, and provide samples, data and expertise from Sweden’s largest microbiology laboratory. In addition, the structure is part of Genomic Medicine Sweden (GMS) which is ensuring the coordination with other national efforts.

The laboratory has established large-scale WGS of SARS-CoV-2. To-date 40,000+ samples sequenced and reported to [SMInet](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/overvakning-och-rapportering/sminet/) and [GISAID](https://gisaid.org/) (capacity 500 WGS/week). The RAPI-SEQ project has generated several papers on SARS-CoV-2 variants and neutralisation.

Metagenomics has been established as a clinical diagnostic tool as well as preparedness for rapid identification of new pandemic threats.

This project involves two subprojects; taxprofiler and GMS Artic. Please see the [taxprofiler](/resources/taxprofiler/) and [GMS Artic](/resources/gms-artic/) pages for more information (including information on access and usage).

**Specific project aims:**

1. An infrastructure for sample handling and sequencing, with access to BSL-2/3 labs, metadata, automation, etc.
2. High-capacity, short-read and long-read sequencing for rapid continuous surveillance of
   emerging SARS-CoV-2 variants linked with capacity for functional tests.
3. Establishment of a metagenomic assay for molecular surveillance of respiratory viruses

##### Research findings:

- We have established an infrastructure for NGS-based pandemic preparedness and response based on a collaboration between KUL within the GMCK (Genome Medicine Center Karolinska) structure, Clinical Genomics at SciLifeLab, and Genomic Medicine
Sweden (GMS).

- We have established new clinical diagnostics based on metagenomics which forms the basis for the surveillance and identification of new pandemic threats.

- We have continuously surveilled emerging SARS-CoV-2 variants by large-scale whole genome sequencing. Through an automated workflow for laboratory work and bioinformatics we have sequenced and reported more than 40,000 SARS-CoV-2 sequences
to the SMInet reporting system, the GENSAM platform at the Public Health Agency (FOHM), and the international GISAID sequence repository.

- We have written a report to the Swedish Corona Commission on the early introduction and spread of SARS-CoV-2 in Sweden. The work has also been published in an international scientific journal (Eurosurveillance). In a separate scientific publication, we have reported on a dual SARS-CoV-2 infection that led to transmission of a recombinant variant. See separate report of published scientific papers.

- Co-PI Benjamin Murrell has established neutralization assays and investigated variants replacement, which takes place due to spike mutations that facilitate escape from immunity (neutralizing antibodies). This work has generated several high-profile papers on the neutralization sensitivity of newly emerging SARS-CoV-2 variants. See separate report
of published scientific papers.

- Co-PI Murrell has built a mathematical model to forecast SARS-CoV-2 variant evolution. The model and the results are available at https://github.com/MurrellGroup/lineages and has been used by FOHM and ECDC in their forecasting and evaluation of the pandemic.

- We have establishment of a metagenomic assay for molecular surveillance of respiratory viruses. The method was introduced as a clinical test in March 2023 following extensive validation and is also valuable for discovery of new pandemic threats. The work has been done in collaboration with other GMS nodes and the PLP project G3P.

- We have developed the following publicly available state-of-the-art bioinformatic pipelines in collaboration with other GMS nodes. The pipelines are written in Nextflow DSL-2 workflow manager and they use Docker/Singularity containers.
SARS-CoV-2 typing: https://github.com/genomic-medicine-sweden/gms-artic. Metagenomics: https://github.com/nf-core/taxprofiler and https://github.com/genomic-medicine-sweden/gmsmetapost.

- We have surveilled the circulation of EV-D68 in respiratory specimens. We have discovered and reported on autumn outbreaks of EV-D68 in 2023 and 2024, which were communicated to national and international authorities and networks. For the 2023 season we were first in the world to report and deposit whole genome sequences in Genbank.

##### Impact on prepardness for future pandemics:

Our work and achievements briefly listed above has significantly contributed to molecular surveillance during the COVID-19-pandemic and has been recognized both nationally and internationally. The infrastructure is operative for clinical diagnostics twenty-four hours a day and seven days a week, which is essential for rapid responses to emerging pandemic threats. The bioinformatics pipelines are publicly available and part of the national GMS structure, in which we have a leading role. As illustrated by the SARS-CoV-2 pandemic, capacity and competence for high-throughput sequencing in real-time allows for informed decisions regarding vaccine design, infection control strategies, and antiviral treatments
using monoclonal antibodies. Our scientific publications and other scientific work has provided important information about the epidemiology, immunology and virology of SARS-CoV-2 and EV-D68. The central role of the RAPID-SEQ resource is illustrated by the fact that Prof Albert currently is employed by the government as inquiry chair (särskild utredare) for
the inquiry on pandemic preparedness (Stärkt beredskap inför framtida pandemier, Dir. 2023:106, https://www.regeringen.se/rattsliga-dokument/kommittedirektiv/2023/07/dir.-2023106