---
title: "Rapid typing of emerging variants in a clinical setting"
category: "tdp"
banner: "/resources_thumbs/rapid_typing.jpg"
resource_info:
  name: "Rapid typing of emerging variants in a clinical setting"
  pi: Anders Bergqvist
  host_organisation: Uppsala University Hospital/Akademiska Sjukhuset Clinical Microbiology, Dept. of Medical Sciences, Uppsala University
  contact: "Anders Bergqvist<br>Uppsala University Hospital/Akademiska Sjukhuset Clinical Microbiology, Dept of Medical Sciences, Uppsala University<br>Email: [Anders.bergqvist@medsci.uu.se](mailto:Anders.bergqvist@medsci.uu.se) or [Anders.bergqvist@akademiska.se](mailto:Anders.bergqvist@akademiska.se)"
for_background_table:
  pi: Anders Bergqvist
  pi_affiliation: Uppsala University
  lab: Uppsala University Hospital
---

Throughout the recent SARS-CoV-2 pandemic, it has been evident that a swift response relies on the capacity to quickly establish methodologies for clinical diagnostics and epidemiologic surveillance. The first generation of commercial kits were sold as "Research Use Only", had low capacity, and were often characterised by poor performance. Robust methodologies were, despite tremendous effort from the manufacturers and the frequent use of "Emergency Use" labelling by regulatory agencies, not available until the first wave was almost over. When the virus variants of concern (VOCs) emerged, it was clear that ambitious sequencing programs, although remarkably impressive and invaluable for the generation of big data, were nevertheless insufficient in terms of capacity and sensitivity, and were inherently too slow for clinical usage. Instead, first line surveillance had to be accomplished by simpler methods that focused on critical sites for defining polymorphism, rather than whole genome sequencing. From spring 2021 to summer 2022, we have typed almost all positive samples and monitored the emergence of different VOCs, including B.1.1.7/Alpha, B.617.2/Delta, as well as the Omicron variants BA.1, BA.2 and BA.5. Whilst the resolution is lower, the sensitivity is higher and its focus on relevant mutations mostly provides sufficient information to correctly identify the VOCs. For both contact tracing and clinical decisions regarding type-specific therapy, rapid typing was far superior compared to whole genome sequencing. In this project, we willÂ develop a generalised platform based on focused sequencing techniques combined with serological testing that can be used for the rapid detection and characterisation of emerging variants of any etiologic agent.

#### Continuation funding

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call.
