---
title: "Rapid establishment of comprehensive laboratory pandemic preparedness for enteroviruses - RAPID-SEQ-Entero"
category: "test"
banner: "/resources_thumbs/rapid-seq-entero.jpg"
resource_info:
  name: "Rapid establishment of comprehensive laboratory pandemic preparedness for enteroviruses - RAPID-SEQ-Entero"
  funded_project_title: "Rapid establishment of comprehensive laboratory pandemic preparedness for enteroviruses - RAPID-SEQ-Entero"
  pi: Jan Albert
  host_organisation: "Karolinska Institutet"
  contact: "Jan Albert<br>Professor<br>Email: [jan.albert@ki.se](mailto:jan.albert@ki.se)"
for_background_table:
  pi: Jan Albert
  pi_affiliation: Karolinska Institutet
---

In this proposal we aim to extend our ongoing RAPID-SEQ project on pandemic laboratory preparedness for SARS-CoV-2 and metagenomics to include research and surveillance on enteroviruses, which have the capacity to cause large worldwide outbreaks. Historically, poliomyelitis is the most well-known and feared enterovirus disease, but many other disease outbreaks have been caused enteroviruses. This includes recent reports on neonatal cases with fulminant and fatal hepatitis due to echovirus 11, polio-like acute flaccid myelitis due to enterovirus D68 (EV-D68), myocarditis due to coxsackie B3 and B4 and neonatal sepsis and meningoencephalitis. Despite the serious consequences and threats from enterovirus infections, Sweden lacks a comprehensive program for surveillance of enteroviruses, apart from the WHO mandated surveillance for poliovirus.

This proposal aims to build capacity and preparedness for comprehensive laboratory surveillance of enteroviruses by:

1. Implementing NGS-based methods for partial and whole genome sequencing (WGS) of enteroviruses and rhinoviruses.
2. Performing enterovirus genotyping on respiratory specimens.
3. Analysing (i) genotypes from patients sampled at an intensive care unit vs other unit, (ii) circulating genotypes over time, and (iii) genotypes across age groups.

Our laboratory is excellently suited to take on the above tasks as being the largest clinical microbiology laboratory in Sweden with continuous access to large numbers of patient samples and results from routine diagnostics. In addition, we are the national reference laboratory for non-polio enteroviruses and a partner in the European non-polio enterovirus network (ENPEN). Within ENPEN we have contributed to several studies on EV-D68. In this proposal we will extend this work to other species and types of enteroviruses and rhinoviruses. We will build on the capacity in our current PLP project (RAPID-SEQ) by adapting the workflow for SARS-CoV-2 to another pathogen.

##### Research findings:

- We have established an infrastructure for NGS-based pandemic preparedness and response based on a collaboration between KUL within the GMCK (Genome Medicine Center Karolinska) structure, Clinical Genomics at SciLifeLab, and Genomic Medicine
Sweden (GMS).

- We have established new clinical diagnostics based on metagenomics which forms the basis for the surveillance and identification of new pandemic threats.

- We have continuously surveilled emerging SARS-CoV-2 variants by large-scale whole genome sequencing. Through an automated workflow for laboratory work and bioinformatics we have sequenced and reported more than 40,000 SARS-CoV-2 sequences
to the SMInet reporting system, the GENSAM platform at the Public Health Agency (FOHM), and the international GISAID sequence repository.

- We have written a report to the Swedish Corona Commission on the early introduction and spread of SARS-CoV-2 in Sweden. The work has also been published in an international scientific journal (Eurosurveillance). In a separate scientific publication, we have reported on a dual SARS-CoV-2 infection that led to transmission of a recombinant variant. See separate report of published scientific papers.

- Co-PI Benjamin Murrell has established neutralization assays and investigated variants replacement, which takes place due to spike mutations that facilitate escape from immunity (neutralizing antibodies). This work has generated several high-profile papers on the neutralization sensitivity of newly emerging SARS-CoV-2 variants. See separate report
of published scientific papers.

- Co-PI Murrell has built a mathematical model to forecast SARS-CoV-2 variant evolution. The model and the results are available at https://github.com/MurrellGroup/lineages and has been used by FOHM and ECDC in their forecasting and evaluation of the pandemic.

- We have establishment of a metagenomic assay for molecular surveillance of respiratory viruses. The method was introduced as a clinical test in March 2023 following extensive validation and is also valuable for discovery of new pandemic threats. The work has been done in collaboration with other GMS nodes and the PLP project G3P.

- We have developed the following publicly available state-of-the-art bioinformatic pipelines in collaboration with other GMS nodes. The pipelines are written in Nextflow DSL-2 workflow manager and they use Docker/Singularity containers.
SARS-CoV-2 typing: https://github.com/genomic-medicine-sweden/gms-artic. Metagenomics: https://github.com/nf-core/taxprofiler and https://github.com/genomic-medicine-sweden/gmsmetapost.

- We have surveilled the circulation of EV-D68 in respiratory specimens. We have discovered and reported on autumn outbreaks of EV-D68 in 2023 and 2024, which were communicated to national and international authorities and networks. For the 2023 season we were first in the world to report and deposit whole genome sequences in Genbank.

##### Impact on prepardness for future pandemics:

Our work and achievements briefly listed above has significantly contributed to molecular surveillance during the COVID-19-pandemic and has been recognized both nationally and internationally. The infrastructure is operative for clinical diagnostics twenty-four hours a day and seven days a week, which is essential for rapid responses to emerging pandemic threats. The bioinformatics pipelines are publicly available and part of the national GMS structure, in which we have a leading role. As illustrated by the SARS-CoV-2 pandemic, capacity and competence for high-throughput sequencing in real-time allows for informed decisions regarding vaccine design, infection control strategies, and antiviral treatments
using monoclonal antibodies. Our scientific publications and other scientific work has provided important information about the epidemiology, immunology and virology of SARS-CoV-2 and EV-D68. The central role of the RAPID-SEQ resource is illustrated by the fact that Prof Albert currently is employed by the government as inquiry chair (särskild utredare) for
the inquiry on pandemic preparedness (Stärkt beredskap inför framtida pandemier, Dir. 2023:106, https://www.regeringen.se/rattsliga-dokument/kommittedirektiv/2023/07/dir.-2023106