---
title: "Rapid typing of emerging variants in a clinical setting"
category: "tdp"
resource_info:
    name: "Rapid typing of emerging variants in a clinical setting"
    pi: Anders Bergqvist
    host_organisation: Uppsala University Hospital/Akademiska Sjukhuset Clinical Microbiology, Dept. of Medical Sciences, Uppsala University
    contact: "Anders Bergqvist<br>Uppsala University Hospital/Akademiska Sjukhuset Clinical Microbiology, Dept of Medical Sciences, Uppsala University<br>Email: [Anders.bergqvist@medsci.uu.se](mailto:Anders.bergqvist@medsci.uu.se) or [Anders.bergqvist@akademiska.se](mailto:Anders.bergqvist@akademiska.se)"
---

During the recent SARS-CoV-2 pandemic it has been evident that a swift response relies on the capacity to quickly establish methodologies for clinical diagnostics and epidemiologic surveillance.Whereas the first generation of commercial kits were sold as Research Use Only, had low capacity and often characterized by poor performance, robust methodologies were despite tremendous effort from the manufacturers and frequent use of Emergency Use labeling by regulatory agencies not available until the first wave was almost over. 

When the virus variants of concern (VOCs) emerged, it was clear that ambitious sequencing programs, although remarkably impressive and invaluable for generation of big data, nevertheless were insufficient in terms of capacity and sensitivity and were inherently too slow for clinical usage. Instead, first line surveillance had to be accomplished by simpler methods that focused on critical sites for defining polymorphism rather than whole genome sequencing. 

From spring 2021 to summer 2022, we have typed almost all positive samples and monitored the emergence of different VOCs including B.1.1.7/Alpha, B.617.2/Delta, as well as the Omicron variants BA.1, BA.2 and BA.5. While the resolution is lower, the sensitivity is higher and its focus on relevant mutations mostly provides sufficient information to correctly identify the VOCs. For contact tracing and for clinical decisions regarding type-specific therapy, rapid typing was far superior compared to whole genome sequencing. In this project, we willÂ develop a generalized platform based on focused sequencing techniques combined with serological testing that can be used for rapid detection and characterization of emerging variants of any etiologic agent.
