---
title: "Establishing sequencing-based viral diagnostics for future clinical use: towards pandemic and outbreak preparedness in the clinical laboratory"
category: "tdp"
banner: "/resources_thumbs/seq-viral-diagnostics.jpg"
resource_info:
  name: "Establishing sequencing-based viral diagnostics for future clinical use: towards pandemic and outbreak preparedness in the clinical laboratory"
  pi: Patrik Medstrand
  host_organisation: Clinical Virology, Lund University and Clinical Microbiology, Medicinsk Service, Region Skåne
  contact: "Patrik Medstrand<br>Clinical Virology, Lund University and Clinical Microbiology, Medicinsk Service, Region Skåne<br>Email: [patrik.medstrand@med.lu.se](mailto:patrik.medstrand@med.lu.se)"
for_background_table:
  pi: Patrik Medstrand
  pi_affiliation: Lund University
  lab: Lund University and Region Skåne
---

This proposal builds on the collaboration between the Dept. of Clinical Microbiology at Region Skåne, the Dept. of Translational Medicine, Section of Clinical Virology at Lund University, and Clinical Genomics Lund at SciLifeLab, to develop and disseminate new state-of-the-art viral diagnostics methods for improved Swedish healthcare. This collaboration already successfully established and implemented whole-genome sequencing (WGS) of SARS-CoV-2 during the pandemic (since February 2021).

WGS of pathogens is becoming increasingly important for clinical science and practice. Moreover, as witnessed by the current pandemic, WGS has proven to be indispensable for tracking the dynamics of SARS-CoV-2 and will be similarly crucial for emerging outbreaks. However, the adoption of WGS in clinical diagnostic laboratories is currently limited and very few WGS protocols for other viruses have been validated for clinical use. In virology, WGS is crucial to inform the development of novel treatments and vaccines, and increases the resolution of molecular epidemiology (for example, in nosocomial transmission) and viral genetic variation as exemplified by the current pandemic. We envision that routine establishment of WGS-based diagnostics of viruses in a clinical setting will become increasingly important for patient care and the wider society, which also will contribute to Pandemic Laboratory preparedness. This proposal specifically aims to accelerate the development of laboratory and bioinformatic WGS protocols in a clinical diagnostic setting for a large number of viruses, to train personnel and increase know-how of WGS in the clinical laboratory, and facilitate data sharing.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. With this funding, the capability will establish a protocol for the detection of unknown pathogens (i.e. direct shotgun sequencing of the metagenomic libraries). The overall aim is to adapt versatile approaches towards clinical implementation. Specifically, the capability is using the funding to:

- Optimise a shutgun metagenomic protocol for use in unknown pathogen detection in a clinical setting.
- Adapt methods to reduce background genetic information that is inherently present in clinical specimens.
- Adapt real-time bioinformatic detection using the MinION Oxford Nanopore using a real-time detection protocol.
- Establish the methodology in the clinical laboratory using a workshop series.
