---
title: "Preparedness for tracking resistant and virulent bacterial and fungal pathogens (TRACK-PATH)"
category: "tdp"
banner: "/resources_thumbs/track-path.jpg"
resource_info:
  name: "Preparedness for tracking resistant and virulent bacterial and fungal pathogens (TRACK-PATH)"
  pi: Christian Giske
  host_organisation: Clinical microbiology, Karolinska University Laboratory and Department of Laboratory medicine, Karolinska Institutet
  contact: "Christian Giske<br>Clinical microbiology, Karolinska University Laboratory and Department of Laboratory medicine, Karolinska Institutet<br>Email: [Christian.giske@ki.se](mailto:Christian.giske@ki.se)"
for_background_table:
  pi: Christian Giske
  pi_affiliation: Karolinska Institutet
  lab: Karolinska Institutet
---

The project consists of three parts:

1. Characterisation of microbial strains (bacteria and fungi) with resistance to recently introduced antimicrobials.
2. Characterisation of virulent pathogens causing unexpectedly severe infections.
3. Exploring methods for studying host-pathogen interaction. During the first months of the project a bioinformatician has been recruited to work on transcriptomics and on establishing pipelines for characterisation of virulent pathogens.

We have selected >40 isogenic mutants of _Escherichia coli_, _Klebsiella pneumoniae_, _Pseudomonas aeruginosa_, and _Acinetobacter baumannii_ with resistance to at least one of the two novel antimicrobials cefiderocol and aztreonam-avibactam. The strains are ready to be analysed regarding transcriptomics, as well as for some chromosomal mechanisms of resistance that are not presently mapped in publicly available pipelines. The goal is to be able to study outer membrane porins, penicillin-binding proteins, multidrug efflux pumps, as well as iron transporters of importance for cefiderocol uptake. The work with the pipelines is already advanced.

We have also explored methods for characterising antifungal resistance in _Candida_ spp, in collaboration with the European reference laboratory at Statens Serum Institut, Copenhagen, Denmark. Finally, we have started collecting lower respiratory samples for assessment of host transcriptomics. This is done in close collaboration with the Niklas Björkström laboratory. The goal is to provide _ex vivo_ information of the ongoing host response and how it differs compared to wildtype variants whilst the latter will yield more reductionistic insights into host responses.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. This funding was intended to enable the capability to:

- Provide guidance on optimal usage of novel antibiotics (e.g. cefiderocol).
- Identify outbreaks in a timely manner by implementing pipelines to identify clusters of SLVs and monitor virulence genes.
- Predict and track antifungal resistance.
- Use a large-scale transcriptomics approach to study host transcriptomics in bloodstream and respiratory tract infections to identify host-pathogen biological imprint.
