---
title: "Preparedness for tracking resistant and virulent bacterial and fungal pathogens (TRACK-PATH)"
category: "tdp"
banner: "/resources_thumbs/track-path.jpg"
resource_info:
  name: "Preparedness for tracking resistant and virulent bacterial and fungal pathogens (TRACK-PATH)"
  pi: Christian Giske
  host_organisation: Clinical microbiology, Karolinska University Laboratory and Department of Laboratory medicine, Karolinska Institutet
  contact: "Christian Giske<br>Clinical microbiology, Karolinska University Laboratory and Department of Laboratory medicine, Karolinska Institutet<br>Email: [Christian.giske@ki.se](mailto:Christian.giske@ki.se)"
for_background_table:
  pi: Christian Giske
  pi_affiliation: Karolinska Institutet
  lab: Karolinska Institutet
---

The project consists of three parts:

1. Characterisation of microbial strains (bacteria and fungi) with resistance to recently introduced antimicrobials.
2. Characterisation of virulent pathogens causing unexpectedly severe infections.
3. Exploring methods for studying host-pathogen interaction. During the first months of the project a bioinformatician has been recruited to work on transcriptomics and on establishing pipelines for characterisation of virulent pathogens.

We have selected >40 isogenic mutants of _Escherichia coli_, _Klebsiella pneumoniae_, _Pseudomonas aeruginosa_, and _Acinetobacter baumannii_ with resistance to at least one of the two novel antimicrobials cefiderocol and aztreonam-avibactam. The strains are ready to be analysed regarding transcriptomics, as well as for some chromosomal mechanisms of resistance that are not presently mapped in publicly available pipelines. The goal is to be able to study outer membrane porins, penicillin-binding proteins, multidrug efflux pumps, as well as iron transporters of importance for cefiderocol uptake. The work with the pipelines is already advanced.

We have also explored methods for characterising antifungal resistance in _Candida_ spp, in collaboration with the European reference laboratory at Statens Serum Institut, Copenhagen, Denmark. Finally, we have started collecting lower respiratory samples for assessment of host transcriptomics. This is done in close collaboration with the Niklas Björkström laboratory. The goal is to provide _ex vivo_ information of the ongoing host response and how it differs compared to wildtype variants whilst the latter will yield more reductionistic insights into host responses.

##### Continuation funding:

This capability received funding from the REPLPCM call to continue their work. Please see our <a href="/plp-program-background/#progress-to-date-previous-calls">PLP background information page</a> for more information about the REPLPCM call. This funding was intended to enable the capability to:

- Provide guidance on optimal usage of novel antibiotics (e.g. cefiderocol).
- Identify outbreaks in a timely manner by implementing pipelines to identify clusters of SLVs and monitor virulence genes.
- Predict and track antifungal resistance.
- Use a large-scale transcriptomics approach to study host transcriptomics in bloodstream and respiratory tract infections to identify host-pathogen biological imprint.

##### Research findings:

- Our research has identified key genetic changes responsible for resistance or decreased susceptibility to ATM-AVI, CEF-ZID, and cefiderocol in carbapenemase-producing E. coli (CP-EC). We found that alterations in penicillin-binding proteins (PBPs 2 and 3) were the main drivers of resistance, parHcularly in globally widespread E. coli sequence types (STs) 405, 410, 167, 361, and 648. These findings raise serious concerns, as these resistant strains are already prevalent worldwide. Additionally, we identified new mutations in PBP2 linked to reduced susceptibility to cefepime-zidebactam, shedding light on emerging threats in antimicrobial resistance.
- We are developing tools, focusing on key pathogens like EC and KP. Analysing over 1200 sequenced EC and KP at KUL (Jan 2020 - Jul 2023) revealed prevalent STs, abundance and genetic context of anHbioHc resistance genes, and clusters of related genomes. We focused mainly on developing bioinformatic pipelines for virulence and antimicrobial resistance gene detection in clinical isolates of KP.
- We are in the process of developing a novel approach to predict antifungal resistance and track the spread of fungal infections over time. Our method integrates the MycoSNP tool for variant analysis and identifying genetic mutations linked to drug resistance. To enhance accuracy, we are currently compiling a comprehensive mutation database by reviewing existing literature and leveraging online resources such as the Mycology Antifungal Resistance Database (MARDy). We have identified suitable strain collections of fungal pathogens and participated in the process of developing susceptibility testing criteria for emerging fungal pathogens (Giske is a member of the European Antifungal Susceptibility Testing Commijee (AFST)). 
- In a pilot study, we optimized an RNA extraction method at the Karolinska University Laboratory. Using this method, RNA isolated from 80 patient samples were subjected to RNAseq and analyzed using the pipelines in Clinical Genomics at SciLifeLab. Our analyses showed two sepsis endotypes with distinct types of inflammatory responses. In collaboration with the Genomic Medicine Sweden (GMS), we continue developing a bioinformatic pipeline using RNAseq data from 80 patients to identify sepsis endotypes. Next, integration of patient chart data with publicly available bulk RNAseq data from prior studies on sepsis patients (N>2000) will be done.

##### Impact on prepardness for future pandemics:

Our project provided critical insights into the genetic mutations driving antimicrobial resistance against novel drugs, supporting enhanced AMR surveillance in key clinical pathogens (EC and KP), capable of causing both community and hospital-acquired infections. Additionally, our RNA sequencing work on sepsis endotypes paves the way for personalized treatments, potentially improving patient survival. By enabling early detection of infectious agents along with their virulence and resistance profiles, our findings can guide tailored infection control strategies and optimize antimicrobial stewardship.
