---
title: "Services: genomics & transcriptomics"
menu:
    main:
        name: Services for researchers
        identifier: genomics_transcriptomics_services
        parent: genomics_transcriptomics
        weight: 30
---

## SciLifeLab Genomics Platform

The [SciLifeLab Genomics Platform](https://www.scilifelab.se/infrastructure/?cat_13168) has a number of different facilities offering services to researchers across a range of techniques.

### Clinical Genomics Stockholm (Diagnostics Development)

The Clinical Genomics Stockholm facility has developed a WGS-based workflow for understanding genetic variation, especially in rare disease diagnostics settings. This workflow includes data generation, bioinformatics analysis and interpretation in our custom-developed clinical decision support system Scout.

We believe this workflow may be useful also for COVID-19 projects aiming at understanding host susceptibility. Data generation is to 30x coverage using PCR-free libraries. The bioinformatic analysis is comprehensive and includes calling of SNV, INDEL, CNV, SV, STR and UPDs. Variants are processed using a ranking model prior to being made available for interpretation in Scout. This infrastructure has been applied to analysis of >6000 WGS samples.

The main focus of the Clinical Genomics Stockholm facility and the Diagnostics Development platform is to support transition of NGS into translational and diagnostic settings. Collaborators include research groups, healthcare, governmental agencies, industry and international groups. Please visit [our website](https://www.scilifelab.se/facilities/clinical-genomics-stockholm/) to find out more about our services, and contact us at [support@clinicalgenomics.se](mailto:support@clinicalgenomics.se) to discuss your specific project.

### High Throughput Genome Engineering (HTGE)

The SciLifeLab High Throughput Genome Engineering Facility provides pooled CRISPR screens from library design to data-analysis, ranging from small, custom-made to genome-wide guide libraries.
In the context of SARS-CoV-2 research, the method is particularly useful to study virus-host interactions and identify cellular genes that are required for virus entry, replication, propagation, release, etc.

For more information about our services, please refer to our webpage [https://www.scilifelab.se/HTGE](https://www.scilifelab.se/HTGE) or contact [bernhard.schmierer@scilifelab.se](mailto:bernhard.schmierer@scilifelab.se).

### National Genomics Infrastructure (NGI)

![NGI Logo](/img/logos/ngi-logo.svg#floatright)

At the SciLifeLab [National Genomics Infrastructure (NGI)](https://ngisweden.scilifelab.se/),
we provide a wide range of sequencing technologies and can offer state-of-the-art solutions for
many different types of COVID-19 sequencing projects.

In addition to [instrumentation](https://ngisweden.scilifelab.se/technologies/)
and next-generation sequencing, NGI provides [library preparation](https://ngisweden.scilifelab.se/applications/)
services and support with project planning and [primary bioinformatics analysis](https://ngisweden.scilifelab.se/bioinformatics/).
Many of the sample and experiment types that we work with could be of use in COVID-19 related research,
whether studying the virus itself or its interactions with the host.

The main focus of the NGI is Swedish academic research groups, though we also work with industry and international groups.
Please visit our website [https://ngisweden.scilifelab.se/contact/](https://ngisweden.scilifelab.se/contact/)
to find out more about our services, and contact us at [support@ngisweden.se](mailto:support@ngisweden.se)
to discuss your specific project.

#### Rapid cDNA and direct RNA sequencing of SARS-CoV-2 using Oxford Nanopore

![Oxford Nanopore MinION](/img/MinION.jpg#floatrightbig)

RNA viruses (including SARS-CoV-2, MERS, SARS) continually accumulate changes in their genomes
that can be used to reconstruct the epidemiological processes that drive the spread of the virus.
Most countries with an active outbreak have sequenced viruses from multiple individuals, allowing
tracking of viral clades and geographic origin of regional outbreaks.

The [National Genomics Infrastructure (NGI)](https://ngisweden.scilifelab.se/) has been granted money from [SciLifeLab](https://www.scilifelab.se/covid-19) and the
[Knut & Wallenberg Foundation (KAW)](https://kaw.wallenberg.org/)
to set up a protocol for SARS-CoV-2 sequencing using the fast and portable
[Oxford Nanopore MinION sequencer](https://nanoporetech.com/products/minion).

##### ARTIC sequencing protocol

We intend to set up the already established and validated protocol available through the
[ARTIC network](https://artic.network/ncov-2019).
The ARTIC network is an organization focused on tracking evolution and spread of viral epidemics, making all sequencing protocols open source and enabling data sharing.
There is a validated protocol available for SARS-CoV-2 sequencing, including primers, laboratory protocols, bioinformatics tutorials and datasets.
An important aspect of the ARTIC network is also the sharing of data across the world.
The protocol has been widely used in a number of countries including China, UK and Iceland.

##### Direct RNA sequencing

It is possible to sequence RNA directly using the Oxford Nanopore MinION, without first converting the RNA to cDNA.
We aim to exploit this unique feature to establish protocols enabling direct sequencing of the entire
RNA genome of SARS-CoV-2.
This makes it possible to not only determine the RNA bases of the viral genome,
but also epigenetic RNA base modifications, and if desirable, the host cell RNA repertoire.
Methylation of specific nucleotides in the SARS-CoV-2 genomes have been reported,
but their potential role in virus biology, virus replication or host-pathogen interaction
are unknown and an important area to investigate further.

For more information about the development and availability of these techniques, please
get in touch  at [support@ngisweden.se](mailto:support@ngisweden.se).

![Oxford Nanopore MinION](/img/MinION-2.jpg)
