---
title: Crystal structure of SARS CoV-2 nsp-10 - a potential drug target for COVID-19 # short
title_full: Crystal structure of SARS CoV-2 nsp-10 - a potential drug target for COVID-19 # long
date: 2020-11-03
summary: A study by Rogstam and colleagues published along with the crystal structure of SARS CoV-2 non-structural protein 10 in the Protein Data Bank provides a starting point for structure-guided drug discovery and design.
banner: /highlights/banners/nsp10.jpg
banner_large: /highlights/banners/nsp10_large.jpeg
banner_caption: "Source: PDB 6ZPE, DOI: 10.2210/pdb6ZPE/pdb"
toc: false
topics: [COVID-19, Infectious diseases]
aliases:
    - /news/nsp10_crystal-structure
    - /sv/news/nsp10_crystal-structure
---
COVID-19 caused by the SARS-CoV-2 virus is now a global pandemic and continues to have severe socio-economic consequences in most parts of the world. The genome sequence of SARS CoV-2 was published in early 2020. Functional information about SARS CoV-2 is still not clear but previous work on the closely related SARS virus has provided some information. A large part of the genome is the Open Reading Frame 1ab (ORF1ab) containing 15 non-structural proteins (nsp1-nsp16), and the remaining part encodes four structural proteins (Spike (S), Nucleocapsid (N), Envelope (E) and Membrane (M) and some smaller accessory proteins which may differ between different coronavirus variants. Non-structural protein 10 (nsp10) is a small protein of ca. 140 amino acid residues found exclusively in viruses – but not in prokaryotes or eukaryotes - and has been shown to be involved in RNA replication.

Rogstam et al. from the Lund University [core facility for protein production (LP3)](https://www.biology.lu.se/services/lp3-lund-protein-production-platform), the BioMAX beamline at the [MAX IV Laboratory](https://www.maxiv.lu.se), [European Spallation Source, ESS](https://www.ess.eu), and the [School of Pharmacy, University College London](https://www.ucl.ac.uk/pharmacy/) studied SARS-CoV-2 nsp10 that binds to and stimulates the 3′-to-5′ exoribonuclease and the 2′-O-methlytransferase activities of nsps 14 and 16, respectively. In their recent publication, Rogstam et al. performed biophysical characterization of the unbound form of nsp10 from SARS CoV-2 and also determined the crystal structure to 1.6 Å resolution. The diffraction data for nsp10 were collected at the BioMAX beamline at the MAX IV laboratory under their call for Rapid Access proposals related to Covid-19.

In addition, the researchers compared the crystal structures of SARS CoV-2 nsp10 with its SARS homologue and the complex-bound form with nsp16 from SARS CoV-2. Rogstam et al. suggest that structural insights into nsp10 could be a first step to enable structure-guided drug discovery for this important target. The results may serve as a basis for future development of inhibitors of the viral replication machinery of SARS CoV-2.

The researchers have shared the crystal structures of nsp10 in the Protein Data Bank (PDB) in Europe hosted by EMBL-EBI. One of the authors of the study, Zoë Fisher explains why they shared the data: “As crystallographers in the structural biology community, we rely on the public availability of crystal structure information. The repository for this is the PDB, and it is an important resource for sharing and disseminating data. In this time of rapid scientific discovery around the novel coronavirus, making scientific data freely available is a crucial collaborative tool, allowing others to benefit from work already done in a transparent way. Sharing knowledge and data is an effective way to push the research forward."

The article was published open access in the *International Journal of Molecular Sciences* on October 6, 2020. Data from the paper is openly available as *Supplemental materials* on the journal website.

*The project was funded by ESS for projects related to COVID-19, and it profited from Lund University’s and its faculties' infrastructure investments as support of the Lund Protein Production Platform (LP3).*

#### Article

DOI: [10.3390/ijms21197375](https://doi.org/10.3390/ijms21197375)

Rogstam A, Nyblom M, Christensen S, Sele C, Talibov VO, Lindvall, T., Andersson Rasmussen, A., André, I., Fisher, Z., Knecht, W., Kozielski F. Crystal Structure of Non-Structural Protein 10 from Severe Acute Respiratory Syndrome Coronavirus-2. *Int J Mol Sci* **21** (19) (2020)

#### Data

* [PDB 6ZPE: Nonstructural protein 10 (nsp10) from SARS CoV-2, 1.58Å resolution](https://www.rcsb.org/structure/6ZPE)
* [PDB 6ZCT: Nonstructural protein 10 (nsp10) from SARS CoV-2, 2.55Å resolution](https://www.rcsb.org/structure/6ZCT)
* [Supplementary materials](http://www.mdpi.com/1422-0067/21/19/7375/s1)
