---
title: "New data highlight on GUI-based workflows for SARS-CoV-2 Nanopore sequencing analysis"
date: 2025-07-03
summary: Researchers from Uppsala University Hospital, developed and validated two automated workflows within the GUI-based software Geneious Prime 2022.1.1. Validation data and tools are openly available via GitHub, and the Sequence Read Archive.
banner: /highlights/banners/nanopore_sequencing.png
banner_caption: "Figure 4 of Cumlin et al. 2024."
---

Our [data highlights](/highlights/) showcase research that shares data and/or other research outputs in as FAIR and Open a manner as possible. The latest data highlight, entitled [‘Automated GUI-based workflows improve accessibility of SARS-CoV-2 Nanopore data analysis’](/highlights/nanopore_sequencing/), shows an example of tools that can be repurposed for multiple pathogens, thus contributing to future pandemic preparedness. Here, researchers at Uppsala University Hospital developed and validated two automated workflows within Geneious Prime, a graphical software platform, to streamline SARS-CoV-2 Oxford Nanopore sequencing analysis.

In particular, [Cumlin _et al._, 2024](https://doi.org/10.3390/ijms25126645), provides a practical solution for labs with limited expertise in coding. The [workflows](https://github.com/clinical-genomics-uppsala/Geneious_SARS-CoV-2) perform trimming, alignment, consensus sequence generation, and (in the advanced version) phylogenetic lineage assignment, using [Pangolin](https://cov-lineages.org/resources/pangolin.html) and [Nextclade](https://joss.theoj.org/papers/10.21105/joss.03773) as integrated plugins.

These user-friendly workflows are already deployed in Sweden’s [national SARS-CoV-2 surveillance program](https://doi.org/10.3390/v14030504), and have been validated across Linux, macOS, and Windows operating systems. Importantly, they have reduced analysis time while preserving diagnostic accuracy.

Validation data and tools are openly available via [GitHub](https://github.com/clinical-genomics-uppsala/Geneious_SARS-CoV-2), and the Sequence Read Archive ([Accession number PRJNA1048178](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1048178/)), ensuring others can adopt or adapt the workflows for different pathogens and outbreak scenarios.

To find out more about the study, please read the [data highlight](/highlights/nanopore_sequencing/). If you are interested in having your work in the areas of pandemic preparedness and/or infectious disease promoted on the portal (e.g. with a data highlight like this one), please [contact the portal team](/contact/).
